--- temp.out.norm	2026-05-12 13:22:07.841009379 +0300
+++ tests/out/test16.out.norm	2026-05-12 13:22:07.842009373 +0300
@@ -5,7 +5,7 @@
 OK
 OK
 OK
-avenger 3 DrStrange SpiderMan WandaMaximoff
+avenger 2 DrStrange SpiderMan
 chaos 1 WandaMaximoff
 hero 1 SpiderMan
 multiverse 2 DrStrange SpiderMan
@@ -13,7 +13,7 @@
 webslinger 1 SpiderMan
 witch 1 WandaMaximoff
 OK
-avenger 3 DrStrange SpiderMan WandaMaximoff
+avenger 2 DrStrange SpiderMan
 chaos 1 WandaMaximoff
 hero 1 SpiderMan
 multiverse 2 DrStrange SpiderMan
