--- temp.out.norm	2026-05-13 10:49:50.805463169 +0300
+++ tests/out/test79.out.norm	2026-05-13 10:49:50.807463211 +0300
@@ -3,7 +3,7 @@
 OK
 OK
 1 c2
-2 c1 c0
+2 c1 c2
 OK
 OK
 3 c0 c2 c4
@@ -13,7 +13,7 @@
 OK
 OK
 2 c2 c5
-2 c2 c2
+2 c2 c5
 2 c0 c3
 4 c0 c1 c2 c5
 2 c1 c2
@@ -89,12 +89,12 @@
 zbor 2 c2 c5
 OK
 OK
-1 (null)
+1 c3
 1 c3
 OK
 OK
 7 c0 c1 c2 c3 c5 c6 c7
-3 c2 c3 c3
+3 c2 c6 c3
 2 c5 c7
 2 c6 c7
 EMPTY
@@ -102,7 +102,7 @@
 1 c7
 OK
 1 c5
-1 (null)
+1 c7
 OK
 abis 1 c1
 algoritm 1 c1
@@ -187,13 +187,13 @@
 2 c2 c8
 OK
 OK
-2 c5 c5
-2 c2 c2
+2 c7 c5
+2 c2 c3
 4 c1 c2 c5 c7
 OK
 NOT FOUND
 7 c1 c10 c2 c3 c5 c7 c9
-1 (null)
+1 c3
 abis 1 c1
 algoritm 2 c1 c10
 ametist 2 c10 c7
