--- temp.out.norm	2026-05-13 10:49:50.671080145 +0300
+++ tests/out/test70.out.norm	2026-05-13 10:49:50.679784307 +0300
@@ -3,7 +3,7 @@
 OK
 OK
 OK
-3 doc1 doc3 doc3
+3 doc1 doc4 doc3
 adancime 2 doc0 doc3
 adrenalina 4 doc1 doc2 doc3 doc4
 aer 2 doc1 doc3
@@ -154,7 +154,7 @@
 zapada 2 doc1 doc3
 zgomot 4 doc1 doc2 doc3 doc4
 OK
-4 doc1 doc4 doc3 doc3
+4 doc1 doc4 doc5 doc3
 4 doc1 doc3 doc4 doc5
 OK
 5 doc1 doc3 doc4 doc5 doc6
@@ -311,10 +311,10 @@
 OK
 OK
 OK
-3 doc0 doc6 doc5
-5 doc4 doc6 doc5 doc3 doc3
+3 doc0 doc7 doc6
+5 doc4 doc7 doc6 doc5 doc3
 OK
-5 doc2 doc4 doc6 doc5 doc3
+5 doc2 doc4 doc8 doc6 doc5
 adancime 6 doc0 doc3 doc5 doc6 doc7 doc8
 adrenalina 6 doc2 doc3 doc4 doc5 doc6 doc8
 aer 5 doc3 doc5 doc6 doc7 doc8
@@ -623,10 +623,10 @@
 zgomot 7 doc2 doc3 doc4 doc5 doc6 doc7 doc8
 7 doc2 doc3 doc4 doc5 doc6 doc7 doc8
 OK
-7 doc4 doc0 doc7 doc6 doc5 doc3 doc3
-2 doc4 doc8
+7 doc4 doc8 doc0 doc7 doc6 doc5 doc3
+2 doc9 doc4
 OK
-3 doc4 doc8 doc7
+3 doc9 doc4 doc8
 1 doc4
 OK
 7 doc10 doc11 doc3 doc4 doc5 doc6 doc9
@@ -1547,9 +1547,9 @@
 vis 11 doc10 doc11 doc13 doc14 doc16 doc17 doc3 doc4 doc5 doc6 doc7
 zapada 10 doc10 doc11 doc12 doc15 doc16 doc17 doc3 doc5 doc6 doc7
 zgomot 12 doc10 doc11 doc13 doc14 doc15 doc16 doc2 doc3 doc4 doc5 doc6 doc7
-5 doc17 doc4 doc11 doc6 doc5
+5 doc17 doc4 doc11 doc7 doc6
 10 doc10 doc12 doc13 doc14 doc16 doc2 doc3 doc4 doc5 doc6
-7 doc2 doc13 doc11 doc16 doc6 doc5 doc3
+7 doc2 doc13 doc11 doc16 doc7 doc6 doc5
 OK
 OK
 3 doc2 doc17 doc13
