--- temp.out.norm	2026-05-12 19:21:31.911185626 +0300
+++ tests/out/test65.out.norm	2026-05-12 19:21:31.917611780 +0300
@@ -6,7 +6,6 @@
 OK
 OK
 3 doc0 doc2 doc3
-EMPTY
 OK
 OK
 adancime 2 doc0 doc5
@@ -160,7 +159,6 @@
 zapada 4 doc0 doc2 doc3 doc5
 zgomot 4 doc0 doc1 doc2 doc5
 5 doc0 doc1 doc2 doc3 doc5
-EMPTY
 OK
 OK
 NOT FOUND
@@ -467,12 +465,10 @@
 zapada 3 doc0 doc3 doc5
 zgomot 3 doc0 doc1 doc5
 3 doc0 doc3 doc5
-EMPTY
 OK
 OK
 OK
 5 doc0 doc1 doc3 doc5 doc6
-EMPTY
 OK
 OK
 OK
@@ -628,7 +624,6 @@
 zapada 5 doc0 doc3 doc5 doc6 doc7
 zgomot 5 doc0 doc1 doc5 doc6 doc7
 7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
-EMPTY
 OK
 OK
 adancime 4 doc0 doc5 doc6 doc7
@@ -784,10 +779,8 @@
 OK
 OK
 7 doc0 doc3 doc4 doc5 doc6 doc7 doc8
-EMPTY
 OK
 3 doc1 doc7 doc8
-EMPTY
 NOT FOUND
 OK
 adancime 5 doc0 doc5 doc6 doc7 doc8
@@ -941,7 +934,6 @@
 zapada 6 doc0 doc3 doc5 doc6 doc7 doc8
 zgomot 6 doc0 doc1 doc5 doc6 doc7 doc8
 5 doc0 doc4 doc5 doc7 doc8
-EMPTY
 OK
 OK
 NOT FOUND
@@ -1097,12 +1089,10 @@
 zapada 6 doc0 doc3 doc5 doc6 doc7 doc8
 zgomot 6 doc0 doc1 doc5 doc6 doc7 doc8
 3 doc0 doc5 doc7
-EMPTY
 NOT FOUND
 OK
 NOT FOUND
 5 doc0 doc3 doc5 doc7 doc8
-EMPTY
 NOT FOUND
 OK
 adancime 5 doc0 doc5 doc6 doc7 doc8
@@ -1407,15 +1397,12 @@
 zapada 6 doc0 doc3 doc5 doc6 doc7 doc8
 zgomot 5 doc0 doc5 doc6 doc7 doc8
 4 doc3 doc5 doc7 doc8
-EMPTY
 OK
 4 doc0 doc3 doc5 doc7
-EMPTY
 OK
 OK
 NOT FOUND
 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
-EMPTY
 OK
 OK
 adancime 6 doc0 doc5 doc6 doc7 doc8 doc9
@@ -1569,20 +1556,16 @@
 zapada 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
 zgomot 6 doc0 doc5 doc6 doc7 doc8 doc9
 5 doc0 doc5 doc7 doc8 doc9
-EMPTY
 OK
 OK
 OK
 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
-EMPTY
 OK
 OK
 OK
 6 doc0 doc3 doc5 doc7 doc8 doc9
-EMPTY
 NOT FOUND
 8 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc9
-EMPTY
 OK
 NOT FOUND
 adancime 7 doc0 doc11 doc5 doc6 doc7 doc8 doc9
@@ -1736,20 +1719,16 @@
 zapada 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
 zgomot 7 doc0 doc11 doc5 doc6 doc7 doc8 doc9
 8 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc9
-EMPTY
 OK
 OK
 OK
 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
-EMPTY
 OK
 NOT FOUND
 NOT FOUND
 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
-EMPTY
 OK
 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
-EMPTY
 OK
 OK
 adancime 6 doc0 doc11 doc5 doc7 doc8 doc9
@@ -2205,13 +2184,10 @@
 zapada 6 doc0 doc3 doc5 doc7 doc8 doc9
 zgomot 6 doc0 doc11 doc5 doc7 doc8 doc9
 4 doc10 doc11 doc7 doc8
-EMPTY
 OK
 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
-EMPTY
 OK
 6 doc0 doc10 doc11 doc7 doc8 doc9
-EMPTY
 OK
 adancime 6 doc0 doc11 doc5 doc7 doc8 doc9
 adrenalina 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
