--- temp.out.norm	2026-05-12 11:27:06.529640511 +0300
+++ tests/out/test30.out.norm	2026-05-12 11:27:06.530182266 +0300
@@ -5,3 +5,17 @@
 OK
 OK
 OK
+OK
+4 Denji HayakawaAki MakimaSan PowerBlood
+3 MakimaSan Denji PowerBlood
+2 MakimaSan Denji
+OK
+3 Denji HayakawaAki MakimaSan
+4 MakimaSan Denji PowerBlood CosmosAngel
+angel 1 CosmosAngel
+blood 1 PowerBlood
+chainsaw 1 Denji
+control 1 MakimaSan
+devil 5 CosmosAngel Denji HayakawaAki MakimaSan PowerBlood
+fiend 2 Denji PowerBlood
+hunter 3 Denji HayakawaAki MakimaSan
