--- temp.out.norm	2026-05-12 10:39:30.178904457 +0300
+++ tests/out/test23.out.norm	2026-05-12 10:39:30.180904431 +0300
@@ -5,10 +5,22 @@
 OK
 OK
 OK
-adventurer 4 ChloeFrager ElenaFisher NathanDrake SamDrake
+OK
+OK
+adventurer 5 ChloeFrager ElenaFisher NathanDrake SamDrake SullyMcaffrey
+hunter 1 NathanDrake
+journalist 1 ElenaFisher
+relic 2 ChloeFrager NathanDrake
+smuggler 1 SullyMcaffrey
+thief 1 SamDrake
+treasure 2 NathanDrake SullyMcaffrey
+5 ChloeFrager ElenaFisher NathanDrake SamDrake SullyMcaffrey
+2 NathanDrake SullyMcaffrey
+OK
+adventurer 5 ChloeFrager ElenaFisher NathanDrake SamDrake SullyMcaffrey
 hunter 1 NathanDrake
 journalist 1 ElenaFisher
-relic 1 ChloeFrager
+relic 1 NathanDrake
 smuggler 1 SullyMcaffrey
 thief 1 SamDrake
 treasure 2 NathanDrake SullyMcaffrey
