--- temp.out.norm	2026-05-12 13:22:07.867009229 +0300
+++ tests/out/test18.out.norm	2026-05-12 13:22:07.869009218 +0300
@@ -8,7 +8,6 @@
 owl 1 Philippa
 school 2 EskelWitcher LambertWitcher
 sorceress 1 Philippa
-spy 1 Philippa
 swords 1 LambertWitcher
 witcher 2 EskelWitcher LambertWitcher
 wolf 2 EskelWitcher LambertWitcher
@@ -17,7 +16,6 @@
 owl 1 Philippa
 school 2 EskelWitcher LambertWitcher
 sorceress 1 Philippa
-spy 1 Philippa
 swords 1 LambertWitcher
 vampire 1 Regis
 witcher 3 EskelWitcher LambertWitcher Regis
