--- temp.out.norm	2026-05-12 11:13:57.336673957 +0300
+++ tests/out/test47.out.norm	2026-05-12 11:13:57.338674052 +0300
@@ -101,7 +101,6 @@
 canapea 3 f_lung_1 f_lung_2 f_lung_3
 canar 3 f_lung_1 f_lung_2 f_lung_3
 nou 3 f_lung_1 f_lung_2 f_lung_3
-EMPTY
 OK
 acas 2 f_lung_2 f_lung_3
 albastru 2 f_lung_2 f_lung_3
@@ -203,7 +202,6 @@
 canapea 2 f_lung_2 f_lung_3
 canar 2 f_lung_2 f_lung_3
 nou 2 f_lung_2 f_lung_3
-EMPTY
 OK
 acas 1 f_lung_2
 albastru 1 f_lung_2
@@ -305,4 +303,3 @@
 canapea 1 f_lung_2
 canar 1 f_lung_2
 nou 1 f_lung_2
-EMPTY
