--- temp.out.norm	2026-05-12 11:13:57.359675049 +0300
+++ tests/out/test49.out.norm	2026-05-12 11:13:57.360675097 +0300
@@ -103,7 +103,6 @@
 vacanta 1 f15
 vara 1 f15
 vers 1 f13
-EMPTY
 OK
 OK
 OK
@@ -174,7 +173,6 @@
 universitate 1 f10
 vacanta 1 f15
 vara 1 f15
-EMPTY
 NOT FOUND
 OK
 OK
@@ -223,7 +221,6 @@
 tabla 1 f14
 vacanta 1 f15
 vara 1 f15
-EMPTY
 OK
 OK
 absente 2 f1 f5
@@ -266,7 +263,6 @@
 tabla 1 f14
 vacanta 1 f15
 vara 1 f15
-EMPTY
 NOT FOUND
 NOT FOUND
 NOT FOUND
@@ -310,4 +306,3 @@
 tabla 1 f14
 vacanta 1 f15
 vara 1 f15
-EMPTY
