--- temp.out.norm	2026-05-13 10:49:50.632459526 +0300
+++ tests/out/test68.out.norm	2026-05-13 10:49:50.633459547 +0300
@@ -22,12 +22,12 @@
 OK
 OK
 OK
-4 inception matrix avatar avatar
-2 morometii gladiatorul
-5 matrix morometii gladiatorul dacii dacii
-1 columna
+4 inception matrix avatar tenet
+2 titanic morometii
+5 titanic matrix morometii gladiatorul dacii
+1 gladiatorul
 OK
-2 gladiatorul gladiatorul
+2 titanic gladiatorul
 1 columna
 4 dacii gladiatorul matrix titanic
 actiune 1 matrix
