--- temp.out.norm	2026-05-12 11:27:06.571181014 +0300
+++ tests/out/test34.out.norm	2026-05-12 11:27:06.572180984 +0300
@@ -5,3 +5,16 @@
 OK
 OK
 OK
+OK
+4 AltairIbnLaAhad ConnorKenway EdwardKenway EzioAuditore
+2 EzioAuditore AltairIbnLaAhad
+4 AltairIbnLaAhad ConnorKenway EdwardKenway EzioAuditore
+3 AltairIbnLaAhad ConnorKenway EzioAuditore
+1 ConnorKenway
+assassin 4 AltairIbnLaAhad ConnorKenway EdwardKenway EzioAuditore
+brotherhood 3 AltairIbnLaAhad ConnorKenway EzioAuditore
+crusades 1 AltairIbnLaAhad
+mohawk 1 ConnorKenway
+pirate 1 EdwardKenway
+renaissance 1 EzioAuditore
+revolution 1 ConnorKenway
