--- temp.out.norm	2026-05-12 10:39:30.669898031 +0300
+++ tests/out/test49.out.norm	2026-05-12 10:39:30.672897992 +0300
@@ -173,12 +173,14 @@
 universitate 1 f10
 vacanta 1 f15
 vara 1 f15
+NOT FOUND
 OK
 OK
 OK
 OK
 OK
 OK
+NOT FOUND
 absente 1 f5
 aplauze 1 f20
 autobuz 1 f18
@@ -261,6 +263,9 @@
 tabla 1 f14
 vacanta 1 f15
 vara 1 f15
+NOT FOUND
+NOT FOUND
+NOT FOUND
 absente 2 f1 f5
 aplauze 1 f20
 autobuz 1 f18
