--- temp.out.norm	2026-05-12 09:14:43.572279304 +0300
+++ tests/out/test05.out.norm	2026-05-12 09:14:43.574279288 +0300
@@ -4,18 +4,18 @@
 OK
 OK
 boston 1 Joel
-fireflies 3 Riley Marlene Joel
-immune 2 Riley Ellie
+fireflies 3 Joel Marlene Riley
+immune 2 Ellie Riley
 infected 1 Ellie
 leader 1 Marlene
 smuggler 1 Tess
-survivor 3 Tess Joel Ellie
+survivor 3 Ellie Joel Tess
 OK
 boston 1 Joel
-fireflies 3 Riley Marlene Joel
-immune 2 Riley Ellie
+fireflies 3 Joel Marlene Riley
+immune 2 Ellie Riley
 infected 1 Ellie
 jackson 1 Tommy
 leader 1 Marlene
 smuggler 1 Tess
-survivor 4 Tommy Tess Joel Ellie
+survivor 4 Ellie Joel Tess Tommy
