--- temp.out.norm	2026-05-12 20:22:23.626233148 +0300
+++ tests/out/test27.out.norm	2026-05-12 20:22:23.627233140 +0300
@@ -7,3 +7,13 @@
 OK
 OK
 3 BoydStevens EthanMatthews Victor
+3 BoydStevens Victor EthanMatthews
+2 BoydStevens Victor
+OK
+4 BoydStevens EthanMatthews SaraMyers Victor
+colony 3 BoydStevens EthanMatthews Victor
+mystery 4 BoydStevens EthanMatthews SaraMyers Victor
+newcomer 2 EthanMatthews JimMatthews
+sheriff 1 BoydStevens
+survivor 2 BoydStevens Victor
+threat 1 SaraMyers
