--- temp.out.norm	2026-05-12 20:22:24.329227663 +0300
+++ tests/out/test80.out.norm	2026-05-12 20:22:24.331227647 +0300
@@ -1,13 +1,24 @@
 OK
 OK
 OK
+2 d0 d1
+2 d0 d2
 OK
 OK
+EMPTY
+3 d0 d2 d3
+EMPTY
 OK
+EMPTY
 3 d0 d2 d3
+2 d0 d3
+1 d2
 OK
 OK
 OK
+2 d0 d4
+1 d4
+2 d2 d4
 OK
 NOT FOUND
 OK
@@ -85,10 +96,15 @@
 vultur 1 d2
 zambet 1 d4
 zbor 1 d4
+2 d0 d4
+3 d0 d5 d4
+4 d0 d2 d4 d5
 OK
 NOT FOUND
+3 d2 d4 d6
 OK
 OK
+EMPTY
 4 d0 d4 d5 d7
 abis 3 d0 d5 d7
 algoritm 4 d2 d4 d5 d7
@@ -164,7 +180,108 @@
 vultur 1 d2
 zambet 1 d4
 zbor 1 d4
+1 d4
+EMPTY
 OK
 4 d2 d4 d6 d7
+EMPTY
 2 d2 d7
 OK
+3 d0 d2 d5
+4 d0 d4 d5 d7
+OK
+OK
+4 d0 d5 d6 d8
+1 d5
+OK
+4 d0 d2 d5 d8
+OK
+3 d0 d5 d8
+EMPTY
+OK
+OK
+3 d0 d7 d8
+2 d0 d5
+5 d0 d2 d5 d7 d8
+OK
+4 d0 d5 d7 d8
+4 d0 d2 d5 d7
+2 d0 d5
+OK
+OK
+OK
+OK
+EMPTY
+EMPTY
+abis 4 d0 d5 d7 d8
+algoritm 4 d2 d5 d7 d8
+ametist 5 d0 d5 d7 d8 d9
+amurg 2 d0 d8
+atlas 2 d0 d7
+azur 3 d2 d5 d8
+brad 2 d2 d8
+busola 3 d0 d7 d8
+cafea 2 d0 d2
+castel 3 d0 d5 d8
+ceara 2 d0 d5
+chitara 3 d0 d7 d8
+coliba 2 d0 d5
+cometa 3 d0 d5 d8
+coral 4 d0 d5 d8 d9
+cristal 2 d5 d8
+cuantic 3 d5 d7 d8
+efemer 2 d2 d7
+elixir 3 d0 d5 d8
+fantana 3 d2 d5 d7
+flacara 3 d0 d2 d8
+fluture 3 d0 d2 d8
+freamat 2 d0 d5
+fulger 4 d2 d5 d7 d8
+galaxie 4 d0 d2 d7 d8
+izvor 2 d2 d8
+jar 3 d2 d7 d8
+labirint 2 d0 d8
+lup 3 d0 d5 d7
+mar 2 d0 d9
+margean 2 d2 d7
+mesteacan 3 d2 d7 d8
+munte 2 d2 d7
+nectar 3 d5 d7 d8
+nisip 3 d0 d5 d7
+nor 2 d0 d8
+ocean 5 d0 d2 d7 d8 d9
+oglinda 3 d0 d2 d8
+orizont 4 d0 d2 d5 d8
+padure 5 d0 d2 d5 d7 d9
+pasare 4 d0 d5 d7 d9
+penita 3 d5 d7 d8
+penumbra 2 d5 d7
+pergament 2 d2 d8
+pian 2 d5 d8
+praf 3 d0 d5 d8
+prisma 3 d2 d7 d8
+randunica 3 d2 d7 d8
+roua 2 d0 d5
+rucsac 3 d2 d5 d9
+sarma 5 d0 d2 d5 d8 d9
+scanteie 4 d0 d5 d7 d8
+sclipici 4 d0 d5 d8 d9
+scoica 2 d5 d9
+sfera 5 d0 d2 d7 d8 d9
+sidef 4 d0 d2 d5 d8
+smarald 5 d0 d2 d5 d7 d9
+spectru 4 d2 d5 d7 d8
+spirala 3 d0 d2 d7
+stea 2 d0 d8
+stejar 2 d0 d2
+tacere 3 d0 d2 d5
+telescop 3 d0 d2 d8
+timp 3 d0 d7 d8
+umbra 2 d0 d5
+univers 1 d0
+vant 1 d8
+vis 3 d0 d2 d8
+viteza 3 d0 d5 d8
+vulcan 3 d5 d7 d8
+vultur 1 d2
+zbor 1 d8
