--- temp.out.norm	2026-05-12 11:18:07.250046654 +0300
+++ tests/out/test55.out.norm	2026-05-12 11:18:07.252046756 +0300
@@ -27,7 +27,7 @@
 OK
 afaceri 1 doc5
 bocanci 1 doc4
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 ceata 1 doc3
 munte 1 doc1
 palmieri 1 doc0
@@ -38,21 +38,21 @@
 OK
 NOT FOUND
 afaceri 1 doc5
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 munte 1 doc1
 palmieri 1 doc0
 prapastie 1 doc6
 OK
 NOT FOUND
 afaceri 1 doc5
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 munte 1 doc1
 palmieri 1 doc0
 parcare 1 doc0
 prapastie 1 doc6
 OK
 afaceri 1 doc5
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 formatiuni 1 doc5
 munte 1 doc1
 palmieri 1 doc0
@@ -63,7 +63,7 @@
 OK
 afaceri 1 doc5
 ancora 1 doc0
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 formatiuni 1 doc5
 munte 1 doc1
 palmieri 1 doc0
@@ -82,7 +82,7 @@
 OK
 afaceri 1 doc5
 ancora 1 doc0
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 comert 1 doc5
 dig 1 doc8
 formatiuni 1 doc5
@@ -103,7 +103,7 @@
 OK
 afaceri 2 doc5 doc9
 ancora 1 doc0
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 comert 1 doc5
 dig 1 doc8
 formatiuni 1 doc5
@@ -125,7 +125,7 @@
 aer 1 doc9
 afaceri 2 doc5 doc9
 ancora 1 doc0
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 comert 1 doc5
 dig 1 doc8
 formatiuni 1 doc5
@@ -148,7 +148,7 @@
 aer 1 doc9
 afaceri 2 doc5 doc9
 ancora 1 doc0
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 cald 1 doc6
 comert 1 doc5
 dig 1 doc8
@@ -165,7 +165,7 @@
 prapastie 1 doc6
 sezlong 1 doc8
 sticla 1 doc11
-umbrela 2 doc6 doc10
+umbrela 2 doc10 doc6
 violet 1 doc8
 OK
 OK
@@ -175,7 +175,7 @@
 aer 1 doc9
 afaceri 2 doc5 doc9
 ancora 2 doc0 doc12
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 cald 1 doc6
 comert 1 doc5
 dig 1 doc8
@@ -192,7 +192,7 @@
 sezlong 1 doc8
 sticla 1 doc11
 traseu 1 doc10
-umbrela 2 doc6 doc10
+umbrela 2 doc10 doc6
 violet 1 doc8
 NOT FOUND
 OK
@@ -226,7 +226,7 @@
 aer 1 doc9
 afaceri 1 doc9
 ancora 2 doc0 doc12
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 cald 1 doc6
 campie 2 doc11 doc15
 comert 1 doc6
@@ -256,7 +256,7 @@
 aer 1 doc9
 afaceri 1 doc9
 ancora 2 doc0 doc12
-brad 2 doc1 doc0
+brad 2 doc0 doc1
 cald 1 doc6
 campie 2 doc11 doc15
 comert 1 doc6
@@ -274,7 +274,7 @@
 plaja 3 doc18 doc21 doc8
 poze 1 doc6
 prapastie 1 doc6
-rasarit 2 doc12 doc0
+rasarit 2 doc0 doc12
 sezlong 1 doc8
 stanca 1 doc14
 sticla 1 doc11
