--- temp.out.norm	2026-05-12 11:13:57.315622001 +0300
+++ tests/out/test45.out.norm	2026-05-12 11:13:57.317673055 +0300
@@ -155,7 +155,6 @@
 vis 3 doc0 doc6 doc7
 zapada 3 doc3 doc4 doc6
 zgomot 2 doc4 doc7
-EMPTY
 OK
 OK
 OK
@@ -315,7 +314,6 @@
 vis 7 doc0 doc10 doc14 doc16 doc6 doc7 doc9
 zapada 8 doc11 doc12 doc13 doc14 doc16 doc3 doc4 doc6
 zgomot 3 doc12 doc4 doc7
-EMPTY
 OK
 OK
 OK
@@ -474,4 +472,3 @@
 vis 10 doc0 doc10 doc14 doc16 doc19 doc21 doc23 doc6 doc7 doc9
 zapada 12 doc11 doc12 doc13 doc14 doc16 doc18 doc20 doc21 doc24 doc3 doc4 doc6
 zgomot 6 doc12 doc19 doc21 doc24 doc4 doc7
-EMPTY
