--- temp.out.norm	2026-05-15 16:27:52.654766232 +0300
+++ tests/out/test14.out.norm	2026-05-15 16:27:52.657765515 +0300
@@ -0,0 +1,22 @@
+OK
+OK
+OK
+OK
+OK
+OK
+OK
+cursed 3 MegumiFushiguro NoboraKugisaki YujiItadori
+divergent 1 YujiItadori
+shikigami 1 MegumiFushiguro
+sorcerer 3 MegumiFushiguro NoboraKugisaki YujiItadori
+tenshadows 1 MegumiFushiguro
+vessel 1 YujiItadori
+OK
+cursed 3 MegumiFushiguro NoboraKugisaki YujiItadori
+divergent 1 YujiItadori
+infinity 1 SatoruGojo
+sensei 1 SatoruGojo
+shikigami 1 MegumiFushiguro
+sorcerer 4 MegumiFushiguro NoboraKugisaki SatoruGojo YujiItadori
+tenshadows 1 MegumiFushiguro
+vessel 1 YujiItadori
