--- temp.out.norm	2026-05-13 10:49:50.645459800 +0300
+++ tests/out/test69.out.norm	2026-05-13 10:49:50.647459842 +0300
@@ -20,7 +20,7 @@
 zapada 2 f1 f6
 3 f1 f2 f6
 2 f1 f6
-3 f1 f2 f2
+3 f1 f6 f2
 OK
 OK
 OK
@@ -39,7 +39,7 @@
 stuf 1 f4
 traseu 1 f2
 zapada 2 f1 f6
-3 f1 f2 f2
+3 f1 f6 f2
 OK
 1 f6
 OK
@@ -74,7 +74,7 @@
 soare 1 f3
 stuf 1 f4
 traseu 1 f2
-2 f5 f5
+2 f7 f5
 OK
 OK
 1 f7
@@ -96,7 +96,7 @@
 OK
 OK
 OK
-1 (null)
+1 f4
 OK
 cabana 1 f6
 iarna 3 f1 f2 f6
