--- temp.out.norm	2026-05-15 16:27:54.924223719 +0300
+++ tests/out/test16.out.norm	2026-05-15 16:27:54.927223002 +0300
@@ -0,0 +1,24 @@
+OK
+OK
+OK
+OK
+OK
+OK
+OK
+avenger 2 DrStrange SpiderMan
+chaos 1 WandaMaximoff
+hero 1 SpiderMan
+multiverse 2 DrStrange SpiderMan
+sorcerer 1 DrStrange
+webslinger 1 SpiderMan
+witch 1 WandaMaximoff
+OK
+avenger 2 DrStrange SpiderMan
+chaos 1 WandaMaximoff
+hero 1 SpiderMan
+multiverse 2 DrStrange SpiderMan
+shield 1 NickFury
+sorcerer 1 DrStrange
+spy 1 NickFury
+webslinger 1 SpiderMan
+witch 1 WandaMaximoff
