--- temp.out.norm	2026-05-15 16:28:32.409626114 +0300
+++ tests/out/test50.out.norm	2026-05-15 16:28:32.413821566 +0300
@@ -705,4 +705,84 @@
 mal 5 doc1 doc15 doc20 doc21 doc7
 mare 9 doc1 doc13 doc15 doc16 doc17 doc19 doc20 doc24 doc3
 marfa 6 doc10 doc13 doc15 doc20 doc4 doc8
-marinari 5 d
\ No newline at end of file
+marinari 5 doc15 doc17 doc20 doc3 doc4
+masina 5 doc11 doc19 doc2 doc20 doc23
+masini 7 doc1 doc11 doc16 doc21 doc23 doc6 doc8
+mister 5 doc1 doc11 doc16 doc23 doc3
+momeala 4 doc13 doc15 doc20 doc3
+motor 5 doc13 doc16 doc20 doc6 doc7
+munte 8 doc15 doc16 doc17 doc19 doc20 doc21 doc23 doc6
+muzica 4 doc11 doc13 doc20 doc8
+natura 5 doc1 doc11 doc2 doc23 doc6
+nisip 5 doc1 doc16 doc19 doc23 doc4
+noapte 6 doc10 doc11 doc13 doc16 doc17 doc6
+nor 4 doc16 doc17 doc21 doc6
+nuca 5 doc17 doc19 doc2 doc20 doc8
+oras 10 doc10 doc13 doc16 doc17 doc21 doc24 doc3 doc4 doc7 doc8
+orientare 3 doc10 doc23 doc3
+otel 6 doc16 doc17 doc20 doc21 doc3 doc8
+padure 5 doc1 doc16 doc21 doc23 doc24
+palmieri 4 doc13 doc16 doc2 doc21
+pamant 6 doc13 doc15 doc16 doc20 doc21 doc24
+parcare 7 doc1 doc11 doc15 doc19 doc20 doc4 doc7
+pasari 4 doc15 doc21 doc24 doc8
+patine 5 doc11 doc15 doc21 doc3 doc8
+patura 3 doc10 doc20 doc7
+peisaj 6 doc11 doc19 doc24 doc4 doc6 doc7
+pelican 8 doc1 doc11 doc13 doc19 doc20 doc21 doc24 doc4
+pescarusi 9 doc10 doc13 doc16 doc19 doc20 doc21 doc3 doc7 doc8
+pescuit 8 doc1 doc10 doc15 doc2 doc20 doc24 doc3 doc8
+peste 5 doc10 doc15 doc16 doc3 doc6
+pestera 6 doc10 doc11 doc13 doc17 doc2 doc8
+picaturi 6 doc11 doc16 doc17 doc19 doc21 doc4
+plaja 7 doc13 doc15 doc19 doc2 doc20 doc21 doc4
+plasa 3 doc11 doc24 doc7
+ploaie 6 doc10 doc16 doc23 doc3 doc6 doc8
+pod 5 doc11 doc13 doc16 doc21 doc7
+poiana 8 doc13 doc17 doc20 doc21 doc3 doc4 doc6 doc8
+port 6 doc10 doc16 doc17 doc24 doc4 doc6
+povesti 7 doc10 doc11 doc15 doc20 doc21 doc23 doc8
+poza 5 doc1 doc16 doc20 doc24 doc3
+poze 7 doc10 doc11 doc20 doc21 doc3 doc4 doc6
+prapastie 6 doc11 doc15 doc16 doc2 doc20 doc24
+prieteni 6 doc10 doc11 doc20 doc21 doc3 doc8
+rasarit 4 doc17 doc2 doc4 doc7
+rece 2 doc15 doc16
+reflectie 5 doc13 doc16 doc23 doc3 doc4
+relaxare 7 doc1 doc13 doc15 doc17 doc20 doc21 doc3
+rucsac 5 doc1 doc10 doc13 doc21 doc7
+salcie 6 doc1 doc10 doc11 doc19 doc21 doc6
+sanie 10 doc1 doc11 doc15 doc16 doc19 doc2 doc23 doc3 doc4 doc7
+sare 3 doc10 doc2 doc21
+schi 1 doc15
+scoici 11 doc10 doc11 doc15 doc17 doc19 doc21 doc24 doc3 doc4 doc7 doc8
+semineu 6 doc10 doc11 doc13 doc3 doc4 doc6
+sezlong 5 doc17 doc19 doc23 doc7 doc8
+soare 7 doc1 doc11 doc13 doc17 doc21 doc4 doc7
+somn 10 doc1 doc10 doc13 doc15 doc16 doc17 doc21 doc23 doc4 doc7
+sport 5 doc10 doc2 doc21 doc5 doc8
+stanca 4 doc1 doc19 doc21 doc8
+stele 10 doc10 doc13 doc19 doc2 doc21 doc23 doc24 doc4 doc5 doc7
+sticla 7 doc1 doc10 doc11 doc16 doc17 doc20 doc8
+stuf 3 doc11 doc13 doc24
+tacere 6 doc10 doc16 doc24 doc4 doc6 doc8
+tehnologie 4 doc2 doc20 doc21 doc4
+timp 2 doc10 doc3
+transport 4 doc16 doc24 doc4 doc8
+traseu 8 doc10 doc19 doc2 doc21 doc23 doc24 doc7 doc8
+trezire 4 doc16 doc23 doc6 doc8
+turist 7 doc1 doc19 doc21 doc23 doc3 doc6 doc8
+umbra 5 doc1 doc10 doc19 doc21 doc23
+umbrela 6 doc10 doc16 doc17 doc2 doc20 doc24
+undita 8 doc1 doc10 doc15 doc20 doc3 doc6 doc7 doc8
+urs 6 doc10 doc11 doc13 doc23 doc24 doc6
+valuri 8 doc10 doc15 doc20 doc21 doc23 doc3 doc6 doc7
+vant 3 doc21 doc23 doc24
+vapor 2 doc10 doc16
+vasle 8 doc1 doc10 doc11 doc19 doc2 doc20 doc24 doc7
+viitor 8 doc10 doc11 doc13 doc15 doc20 doc21 doc24 doc8
+vin 6 doc11 doc13 doc2 doc23 doc3 doc6
+violet 7 doc11 doc13 doc15 doc20 doc6 doc7 doc8
+vis 6 doc1 doc13 doc19 doc24 doc3 doc4
+zapada 2 doc2 doc21
+zgomot 3 doc11 doc20 doc3
