--- temp.out.norm	2026-05-13 10:49:50.042447103 +0300
+++ tests/out/test29.out.norm	2026-05-13 10:49:50.043447124 +0300
@@ -7,8 +7,8 @@
 OK
 OK
 4 MegumiFushiguro NoboraKugisaki SatoruGojo YujiItadori
-3 SatoruGojo MegumiFushiguro NoboraKugisaki
-2 MegumiFushiguro NoboraKugisaki
+3 SatoruGojo YujiItadori MegumiFushiguro
+2 YujiItadori MegumiFushiguro
 OK
 4 MegumiFushiguro NoboraKugisaki RyomenSukuna YujiItadori
 cursed 4 MegumiFushiguro NoboraKugisaki RyomenSukuna YujiItadori
