--- temp.out.norm	2026-05-12 11:27:06.652178543 +0300
+++ tests/out/test40.out.norm	2026-05-12 11:27:06.653178512 +0300
@@ -5,3 +5,15 @@
 OK
 OK
 OK
+OK
+4 DustinHenderson ElevenHopper HopperChief MaxMayfield
+2 DustinHenderson MaxMayfield
+4 DustinHenderson ElevenHopper HopperChief MaxMayfield
+1 ElevenHopper
+2 DustinHenderson MaxMayfield
+chief 1 HopperChief
+hawkins 4 DustinHenderson ElevenHopper HopperChief MaxMayfield
+hero 1 ElevenHopper
+party 2 DustinHenderson MaxMayfield
+psychic 1 ElevenHopper
+upsidedown 1 ElevenHopper
