--- temp.out.norm	2026-05-15 16:31:20.216148952 +0300
+++ tests/out/test40.out.norm	2026-05-15 16:31:20.219148235 +0300
@@ -0,0 +1,19 @@
+OK
+OK
+OK
+OK
+OK
+OK
+OK
+OK
+4 DustinHenderson ElevenHopper HopperChief MaxMayfield
+2 DustinHenderson MaxMayfield
+4 DustinHenderson ElevenHopper HopperChief MaxMayfield
+1 ElevenHopper
+2 DustinHenderson MaxMayfield
+chief 1 HopperChief
+hawkins 4 DustinHenderson ElevenHopper HopperChief MaxMayfield
+hero 1 ElevenHopper
+party 2 DustinHenderson MaxMayfield
+psychic 1 ElevenHopper
+upsidedown 1 ElevenHopper
