--- temp.out.norm	2026-05-12 20:22:23.651232953 +0300
+++ tests/out/test29.out.norm	2026-05-12 20:22:23.652232945 +0300
@@ -7,3 +7,14 @@
 OK
 OK
 4 MegumiFushiguro NoboraKugisaki SatoruGojo YujiItadori
+3 SatoruGojo YujiItadori MegumiFushiguro
+2 YujiItadori MegumiFushiguro
+OK
+4 MegumiFushiguro NoboraKugisaki RyomenSukuna YujiItadori
+cursed 4 MegumiFushiguro NoboraKugisaki RyomenSukuna YujiItadori
+divergent 1 YujiItadori
+infinity 1 SatoruGojo
+king 1 RyomenSukuna
+sensei 1 SatoruGojo
+sorcerer 4 MegumiFushiguro NoboraKugisaki SatoruGojo YujiItadori
+vessel 1 YujiItadori
