--- temp.out.norm	2026-05-15 16:28:31.267535897 +0300
+++ tests/out/test49.out.norm	2026-05-15 16:28:31.269535419 +0300
@@ -293,4 +293,16 @@
 mare 1 f15
 munte 1 f15
 note 2 f1 f5
-olimpiada
\ No newline at end of file
+olimpiada 1 f16
+pix 1 f9
+prezenta 2 f1 f5
+prieten 1 f14
+reguli 2 f19 f21
+scena 1 f20
+scoala 11 f1 f11 f14 f15 f16 f18 f19 f20 f21 f5 f9
+serbare 1 f20
+soare 1 f15
+statie 1 f18
+tabla 1 f14
+vacanta 1 f15
+vara 1 f15
