--- temp.out.norm	2026-05-15 16:27:50.389307786 +0300
+++ tests/out/test12.out.norm	2026-05-15 16:27:50.392307069 +0300
@@ -0,0 +1,17 @@
+OK
+OK
+OK
+OK
+OK
+OK
+colony 3 BoydStevens EthanMatthews TabithaMatthews
+mystery 2 BoydStevens EthanMatthews
+newcomer 2 EthanMatthews TabithaMatthews
+sheriff 1 BoydStevens
+survivor 1 BoydStevens
+OK
+colony 3 BoydStevens EthanMatthews TabithaMatthews
+mystery 2 BoydStevens EthanMatthews
+newcomer 3 EthanMatthews JimMatthews TabithaMatthews
+sheriff 1 BoydStevens
+survivor 2 BoydStevens JimMatthews
