--- temp.out.norm	2026-05-12 11:21:12.656611526 +0300
+++ tests/out/test15.out.norm	2026-05-12 11:21:12.658611630 +0300
@@ -0,0 +1,20 @@
+OK
+OK
+OK
+OK
+OK
+OK
+blood 1 PowerBlood
+chainsaw 1 Denji
+devil 3 Denji HayakawaAki PowerBlood
+fiend 2 Denji PowerBlood
+hunter 2 Denji HayakawaAki
+sword 1 HayakawaAki
+OK
+blood 1 PowerBlood
+chainsaw 1 Denji
+control 1 MakimaSan
+devil 4 Denji HayakawaAki MakimaSan PowerBlood
+fiend 2 Denji PowerBlood
+hunter 3 Denji HayakawaAki MakimaSan
+sword 1 HayakawaAki
