--- temp.out.norm	2026-05-15 16:28:27.827766282 +0300
+++ tests/out/test45.out.norm	2026-05-15 16:28:27.830969256 +0300
@@ -421,4 +421,54 @@
 peste 12 doc0 doc10 doc11 doc14 doc15 doc16 doc19 doc21 doc24 doc5 doc6 doc7
 pestera 8 doc0 doc16 doc17 doc2 doc21 doc24 doc8 doc9
 picaturi 7 doc12 doc15 doc2 doc20 doc23 doc24 doc7
-plaja 5 doc10 doc16 doc5 doc7
\ No newline at end of file
+plaja 5 doc10 doc16 doc5 doc7 doc9
+plasa 5 doc0 doc14 doc19 doc21 doc22
+ploaie 9 doc16 doc17 doc18 doc19 doc24 doc3 doc5 doc6 doc9
+pod 7 doc0 doc12 doc16 doc24 doc4 doc5 doc8
+poiana 9 doc11 doc12 doc13 doc14 doc15 doc17 doc18 doc21 doc8
+port 9 doc12 doc16 doc17 doc19 doc23 doc25 doc3 doc6 doc8
+povesti 8 doc0 doc10 doc18 doc19 doc20 doc3 doc5 doc8
+poza 13 doc0 doc1 doc11 doc12 doc15 doc16 doc19 doc21 doc23 doc4 doc6 doc8 doc9
+poze 9 doc10 doc17 doc19 doc20 doc21 doc3 doc4 doc8 doc9
+prapastie 8 doc1 doc14 doc18 doc2 doc24 doc25 doc5 doc6
+prieteni 7 doc0 doc10 doc11 doc12 doc16 doc7 doc9
+rasarit 5 doc11 doc15 doc25 doc3 doc4
+rece 7 doc12 doc17 doc19 doc2 doc21 doc7 doc8
+reflectie 5 doc14 doc15 doc2 doc20 doc25
+relaxare 8 doc10 doc11 doc12 doc2 doc25 doc4 doc5 doc7
+rucsac 13 doc0 doc16 doc19 doc20 doc21 doc22 doc23 doc24 doc25 doc5 doc6 doc7 doc8
+salcie 9 doc0 doc10 doc14 doc20 doc22 doc24 doc25 doc3 doc8
+sanie 11 doc0 doc12 doc16 doc20 doc22 doc24 doc3 doc6 doc7 doc8 doc9
+sare 12 doc0 doc11 doc13 doc16 doc19 doc20 doc22 doc24 doc25 doc3 doc4 doc7
+schi 12 doc0 doc13 doc18 doc20 doc21 doc22 doc23 doc25 doc4 doc6 doc7 doc9
+scoici 11 doc10 doc14 doc18 doc19 doc2 doc20 doc22 doc25 doc5 doc7 doc8
+semineu 8 doc19 doc20 doc21 doc22 doc3 doc5 doc6 doc9
+sezlong 10 doc0 doc1 doc10 doc12 doc15 doc16 doc20 doc21 doc5 doc9
+soare 9 doc10 doc19 doc20 doc22 doc23 doc5 doc6 doc8 doc9
+somn 7 doc1 doc11 doc16 doc18 doc22 doc25 doc6
+sport 7 doc11 doc19 doc21 doc5 doc6 doc7 doc8
+stanca 7 doc1 doc13 doc15 doc4 doc6 doc8 doc9
+stele 8 doc0 doc10 doc11 doc15 doc17 doc22 doc4 doc8
+sticla 11 doc0 doc1 doc10 doc12 doc13 doc15 doc18 doc20 doc22 doc24 doc8
+stuf 6 doc1 doc10 doc11 doc17 doc2 doc25
+tacere 14 doc0 doc1 doc10 doc11 doc13 doc16 doc20 doc22 doc23 doc25 doc4 doc5 doc6 doc7
+tehnologie 9 doc0 doc1 doc11 doc13 doc15 doc16 doc25 doc4 doc8
+timp 9 doc0 doc10 doc13 doc15 doc21 doc22 doc4 doc7 doc9
+transport 11 doc0 doc11 doc14 doc19 doc20 doc24 doc4 doc5 doc6 doc7 doc8
+traseu 5 doc17 doc19 doc2 doc21 doc7
+trezire 9 doc1 doc11 doc2 doc20 doc22 doc25 doc4 doc7 doc8
+turist 9 doc10 doc11 doc12 doc14 doc20 doc21 doc22 doc3 doc8
+umbra 7 doc0 doc1 doc15 doc18 doc19 doc23 doc7
+umbrela 9 doc1 doc10 doc11 doc17 doc18 doc19 doc20 doc21 doc5
+undita 10 doc0 doc11 doc14 doc17 doc2 doc21 doc22 doc23 doc4 doc8
+urs 8 doc0 doc10 doc13 doc21 doc23 doc24 doc25 doc4
+valuri 6 doc10 doc19 doc2 doc20 doc4 doc9
+vant 9 doc0 doc12 doc14 doc18 doc19 doc20 doc21 doc5 doc9
+vapor 10 doc0 doc1 doc12 doc13 doc15 doc20 doc24 doc25 doc7 doc9
+vasle 9 doc11 doc14 doc16 doc18 doc19 doc22 doc23 doc24 doc6
+viitor 7 doc13 doc20 doc23 doc3 doc5 doc6 doc8
+vin 7 doc16 doc18 doc20 doc21 doc23 doc24 doc8
+violet 10 doc0 doc10 doc13 doc15 doc19 doc20 doc21 doc25 doc5 doc6
+vis 10 doc0 doc10 doc14 doc16 doc19 doc21 doc23 doc6 doc7 doc9
+zapada 12 doc11 doc12 doc13 doc14 doc16 doc18 doc20 doc21 doc24 doc3 doc4 doc6
+zgomot 6 doc12 doc19 doc21 doc24 doc4 doc7
