--- temp.out.norm	2026-05-12 20:22:24.167228927 +0300
+++ tests/out/test68.out.norm	2026-05-12 20:22:24.168228919 +0300
@@ -22,3 +22,21 @@
 OK
 OK
 OK
+4 inception matrix avatar tenet
+2 titanic morometii
+5 titanic matrix morometii gladiatorul dacii
+1 gladiatorul
+OK
+2 titanic gladiatorul
+1 columna
+4 dacii gladiatorul matrix titanic
+actiune 1 matrix
+aventura 1 avatar
+drama 2 gladiatorul titanic
+istorie 2 columna dacii
+mister 1 memento
+premiu 4 dacii gladiatorul matrix titanic
+razboi 3 columna dacii gladiatorul
+romance 1 titanic
+sf 4 avatar inception matrix tenet
+thriller 3 inception memento tenet
