--- temp.out.norm	2026-05-13 10:49:50.011446450 +0300
+++ tests/out/test27.out.norm	2026-05-13 10:49:50.013446492 +0300
@@ -7,8 +7,8 @@
 OK
 OK
 3 BoydStevens EthanMatthews Victor
-3 BoydStevens EthanMatthews EthanMatthews
-2 BoydStevens EthanMatthews
+3 BoydStevens Victor EthanMatthews
+2 BoydStevens Victor
 OK
 4 BoydStevens EthanMatthews SaraMyers Victor
 colony 3 BoydStevens EthanMatthews Victor
