--- temp.out.norm	2026-05-15 16:31:34.581714886 +0300
+++ tests/out/test29.out.norm	2026-05-15 16:31:34.584714169 +0300
@@ -0,0 +1,20 @@
+OK
+OK
+OK
+OK
+OK
+OK
+OK
+OK
+4 MegumiFushiguro NoboraKugisaki SatoruGojo YujiItadori
+3 SatoruGojo YujiItadori MegumiFushiguro
+2 YujiItadori MegumiFushiguro
+OK
+4 MegumiFushiguro NoboraKugisaki RyomenSukuna YujiItadori
+cursed 4 MegumiFushiguro NoboraKugisaki RyomenSukuna YujiItadori
+divergent 1 YujiItadori
+infinity 1 SatoruGojo
+king 1 RyomenSukuna
+sensei 1 SatoruGojo
+sorcerer 4 MegumiFushiguro NoboraKugisaki SatoruGojo YujiItadori
+vessel 1 YujiItadori
