--- temp.out.norm	2026-05-12 10:39:30.215903973 +0300
+++ tests/out/test25.out.norm	2026-05-12 10:39:30.217903947 +0300
@@ -5,8 +5,22 @@
 OK
 OK
 OK
+OK
+OK
 chief 1 HopperChief
 hawkins 5 DustinHenderson ElevenHopper HopperChief MaxMayfield MikeWheeler
+hero 1 ElevenHopper
 party 3 DustinHenderson MaxMayfield MikeWheeler
 psychic 1 ElevenHopper
+survivor 1 HopperChief
+upsidedown 2 ElevenHopper HopperChief
+5 DustinHenderson ElevenHopper HopperChief MaxMayfield MikeWheeler
+2 ElevenHopper HopperChief
+OK
+chief 1 HopperChief
+hawkins 5 DustinHenderson ElevenHopper HopperChief MaxMayfield MikeWheeler
+hero 1 ElevenHopper
+party 2 DustinHenderson MaxMayfield
+psychic 1 ElevenHopper
+survivor 1 HopperChief
 upsidedown 2 ElevenHopper HopperChief
