--- temp.out.norm	2026-05-12 22:56:46.930406330 +0300
+++ tests/out/test40.out.norm	2026-05-12 22:56:46.931406318 +0300
@@ -7,5 +7,13 @@
 OK
 OK
 4 DustinHenderson ElevenHopper HopperChief MaxMayfield
+2 DustinHenderson MaxMayfield
 4 DustinHenderson ElevenHopper HopperChief MaxMayfield
 1 ElevenHopper
+2 DustinHenderson MaxMayfield
+chief 1 HopperChief
+hawkins 4 DustinHenderson ElevenHopper HopperChief MaxMayfield
+hero 1 ElevenHopper
+party 2 DustinHenderson MaxMayfield
+psychic 1 ElevenHopper
+upsidedown 1 ElevenHopper
