--- temp.out.norm	2026-05-12 11:27:06.943706035 +0300
+++ tests/out/test65.out.norm	2026-05-12 11:27:06.950058032 +0300
@@ -5,6 +5,9 @@
 OK
 OK
 OK
+3 doc0 doc2 doc3
+OK
+OK
 adancime 2 doc0 doc5
 adrenalina 3 doc0 doc2 doc3
 aer 6 doc0 doc1 doc2 doc3 doc4 doc5
@@ -45,7 +48,7 @@
 echipa 3 doc0 doc3 doc5
 ecou 5 doc0 doc2 doc3 doc4 doc5
 efort 3 doc0 doc3 doc5
-emotie 4 doc0 doc3 doc4 doc5
+emotie 3 doc3 doc4 doc5
 explorare 6 doc0 doc1 doc2 doc3 doc4 doc5
 faleza 6 doc0 doc1 doc2 doc3 doc4 doc5
 floare 3 doc0 doc3 doc5
@@ -136,7 +139,7 @@
 stuf 5 doc0 doc1 doc3 doc4 doc5
 tacere 4 doc0 doc1 doc3 doc5
 tehnologie 6 doc0 doc1 doc2 doc3 doc4 doc5
-timp 4 doc0 doc2 doc3 doc5
+timp 3 doc2 doc3 doc5
 transport 5 doc0 doc1 doc2 doc3 doc5
 traseu 3 doc0 doc3 doc5
 trezire 4 doc0 doc1 doc3 doc5
@@ -155,8 +158,11 @@
 vis 4 doc0 doc2 doc4 doc5
 zapada 4 doc0 doc2 doc3 doc5
 zgomot 4 doc0 doc1 doc2 doc5
+5 doc0 doc1 doc2 doc3 doc5
 OK
 OK
+NOT FOUND
+NOT FOUND
 adancime 2 doc0 doc5
 adrenalina 2 doc0 doc3
 aer 5 doc0 doc1 doc3 doc4 doc5
@@ -197,7 +203,7 @@
 echipa 3 doc0 doc3 doc5
 ecou 4 doc0 doc3 doc4 doc5
 efort 3 doc0 doc3 doc5
-emotie 4 doc0 doc3 doc4 doc5
+emotie 3 doc3 doc4 doc5
 explorare 5 doc0 doc1 doc3 doc4 doc5
 faleza 5 doc0 doc1 doc3 doc4 doc5
 floare 3 doc0 doc3 doc5
@@ -288,7 +294,7 @@
 stuf 5 doc0 doc1 doc3 doc4 doc5
 tacere 4 doc0 doc1 doc3 doc5
 tehnologie 5 doc0 doc1 doc3 doc4 doc5
-timp 3 doc0 doc3 doc5
+timp 2 doc3 doc5
 transport 4 doc0 doc1 doc3 doc5
 traseu 3 doc0 doc3 doc5
 trezire 4 doc0 doc1 doc3 doc5
@@ -307,6 +313,7 @@
 vis 3 doc0 doc4 doc5
 zapada 3 doc0 doc3 doc5
 zgomot 3 doc0 doc1 doc5
+OK
 adancime 2 doc0 doc5
 adrenalina 2 doc0 doc3
 aer 5 doc0 doc1 doc3 doc4 doc5
@@ -347,7 +354,7 @@
 echipa 3 doc0 doc3 doc5
 ecou 4 doc0 doc3 doc4 doc5
 efort 3 doc0 doc3 doc5
-emotie 4 doc0 doc3 doc4 doc5
+emotie 3 doc3 doc4 doc5
 explorare 5 doc0 doc1 doc3 doc4 doc5
 faleza 5 doc0 doc1 doc3 doc4 doc5
 floare 3 doc0 doc3 doc5
@@ -413,7 +420,7 @@
 poiana 4 doc0 doc1 doc4 doc5
 port 3 doc0 doc3 doc4
 povesti 3 doc0 doc3 doc5
-poza 2 doc0 doc1
+poza 1 doc1
 poze 4 doc0 doc3 doc4 doc5
 prapastie 2 doc0 doc3
 prieteni 4 doc0 doc1 doc3 doc5
@@ -438,7 +445,7 @@
 stuf 5 doc0 doc1 doc3 doc4 doc5
 tacere 4 doc0 doc1 doc3 doc5
 tehnologie 5 doc0 doc1 doc3 doc4 doc5
-timp 3 doc0 doc3 doc5
+timp 2 doc3 doc5
 transport 4 doc0 doc1 doc3 doc5
 traseu 3 doc0 doc3 doc5
 trezire 4 doc0 doc1 doc3 doc5
@@ -457,6 +464,11 @@
 vis 3 doc0 doc4 doc5
 zapada 3 doc0 doc3 doc5
 zgomot 3 doc0 doc1 doc5
+3 doc0 doc3 doc5
+OK
+OK
+OK
+5 doc0 doc1 doc3 doc5 doc6
 OK
 OK
 OK
@@ -501,11 +513,11 @@
 echipa 5 doc0 doc3 doc5 doc6 doc7
 ecou 6 doc0 doc3 doc4 doc5 doc6 doc7
 efort 5 doc0 doc3 doc5 doc6 doc7
-emotie 5 doc0 doc3 doc4 doc5 doc7
+emotie 4 doc3 doc4 doc5 doc7
 explorare 7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
 faleza 6 doc0 doc1 doc3 doc4 doc5 doc7
 floare 5 doc0 doc3 doc5 doc6 doc7
-fluviu 6 doc0 doc3 doc4 doc5 doc6 doc7
+fluviu 5 doc3 doc4 doc5 doc6 doc7
 foc 6 doc0 doc3 doc4 doc5 doc6 doc7
 formatiuni 5 doc0 doc1 doc3 doc5 doc7
 frica 4 doc0 doc4 doc6 doc7
@@ -535,7 +547,7 @@
 masini 6 doc0 doc1 doc3 doc4 doc5 doc7
 mister 5 doc0 doc3 doc5 doc6 doc7
 momeala 5 doc0 doc3 doc4 doc5 doc7
-motor 4 doc0 doc3 doc5 doc7
+motor 3 doc3 doc5 doc7
 munte 5 doc0 doc3 doc4 doc6 doc7
 muzica 5 doc0 doc3 doc4 doc5 doc7
 natura 5 doc1 doc3 doc5 doc6 doc7
@@ -559,7 +571,7 @@
 pescuit 7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
 peste 5 doc0 doc1 doc3 doc5 doc7
 pestera 4 doc0 doc3 doc6 doc7
-picaturi 5 doc0 doc3 doc5 doc6 doc7
+picaturi 4 doc0 doc5 doc6 doc7
 plaja 6 doc0 doc3 doc4 doc5 doc6 doc7
 plasa 4 doc0 doc4 doc5 doc7
 ploaie 5 doc0 doc3 doc5 doc6 doc7
@@ -567,7 +579,7 @@
 poiana 5 doc0 doc1 doc4 doc5 doc7
 port 4 doc0 doc3 doc4 doc7
 povesti 4 doc0 doc3 doc5 doc7
-poza 3 doc0 doc1 doc7
+poza 2 doc1 doc7
 poze 5 doc0 doc3 doc4 doc5 doc7
 prapastie 4 doc0 doc3 doc6 doc7
 prieteni 6 doc0 doc1 doc3 doc5 doc6 doc7
@@ -592,7 +604,7 @@
 stuf 7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
 tacere 6 doc0 doc1 doc3 doc5 doc6 doc7
 tehnologie 7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
-timp 5 doc0 doc3 doc5 doc6 doc7
+timp 4 doc3 doc5 doc6 doc7
 transport 6 doc0 doc1 doc3 doc5 doc6 doc7
 traseu 4 doc0 doc3 doc5 doc7
 trezire 5 doc0 doc1 doc3 doc5 doc7
@@ -611,6 +623,8 @@
 vis 4 doc0 doc4 doc5 doc7
 zapada 5 doc0 doc3 doc5 doc6 doc7
 zgomot 5 doc0 doc1 doc5 doc6 doc7
+7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
+OK
 OK
 adancime 4 doc0 doc5 doc6 doc7
 adrenalina 4 doc0 doc3 doc6 doc7
@@ -652,11 +666,11 @@
 echipa 5 doc0 doc3 doc5 doc6 doc7
 ecou 6 doc0 doc3 doc4 doc5 doc6 doc7
 efort 5 doc0 doc3 doc5 doc6 doc7
-emotie 5 doc0 doc3 doc4 doc5 doc7
+emotie 4 doc3 doc4 doc5 doc7
 explorare 7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
 faleza 6 doc0 doc1 doc3 doc4 doc5 doc7
 floare 5 doc0 doc3 doc5 doc6 doc7
-fluviu 6 doc0 doc3 doc4 doc5 doc6 doc7
+fluviu 5 doc3 doc4 doc5 doc6 doc7
 foc 6 doc0 doc3 doc4 doc5 doc6 doc7
 formatiuni 5 doc0 doc1 doc3 doc5 doc7
 frica 4 doc0 doc4 doc6 doc7
@@ -686,7 +700,7 @@
 masini 6 doc0 doc1 doc3 doc4 doc5 doc7
 mister 5 doc0 doc3 doc5 doc6 doc7
 momeala 5 doc0 doc3 doc4 doc5 doc7
-motor 4 doc0 doc3 doc5 doc7
+motor 3 doc3 doc5 doc7
 munte 5 doc0 doc3 doc4 doc6 doc7
 muzica 5 doc0 doc3 doc4 doc5 doc7
 natura 5 doc1 doc3 doc5 doc6 doc7
@@ -710,7 +724,7 @@
 pescuit 7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
 peste 5 doc0 doc1 doc3 doc5 doc7
 pestera 4 doc0 doc3 doc6 doc7
-picaturi 5 doc0 doc3 doc5 doc6 doc7
+picaturi 4 doc0 doc5 doc6 doc7
 plaja 6 doc0 doc3 doc4 doc5 doc6 doc7
 plasa 4 doc0 doc4 doc5 doc7
 ploaie 5 doc0 doc3 doc5 doc6 doc7
@@ -718,11 +732,11 @@
 poiana 5 doc0 doc1 doc4 doc5 doc7
 port 4 doc0 doc3 doc4 doc7
 povesti 4 doc0 doc3 doc5 doc7
-poza 3 doc0 doc1 doc7
+poza 2 doc1 doc7
 poze 5 doc0 doc3 doc4 doc5 doc7
 prapastie 4 doc0 doc3 doc6 doc7
 prieteni 6 doc0 doc1 doc3 doc5 doc6 doc7
-rasarit 6 doc0 doc1 doc3 doc5 doc6 doc7
+rasarit 5 doc0 doc1 doc3 doc5 doc6
 rece 5 doc0 doc1 doc3 doc5 doc7
 reflectie 5 doc0 doc3 doc5 doc6 doc7
 relaxare 5 doc0 doc3 doc5 doc6 doc7
@@ -743,7 +757,7 @@
 stuf 7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
 tacere 6 doc0 doc1 doc3 doc5 doc6 doc7
 tehnologie 7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
-timp 5 doc0 doc3 doc5 doc6 doc7
+timp 4 doc3 doc5 doc6 doc7
 transport 6 doc0 doc1 doc3 doc5 doc6 doc7
 traseu 4 doc0 doc3 doc5 doc7
 trezire 5 doc0 doc1 doc3 doc5 doc7
@@ -764,7 +778,11 @@
 zgomot 5 doc0 doc1 doc5 doc6 doc7
 OK
 OK
+7 doc0 doc3 doc4 doc5 doc6 doc7 doc8
+OK
+3 doc1 doc7 doc8
 NOT FOUND
+OK
 adancime 5 doc0 doc5 doc6 doc7 doc8
 adrenalina 5 doc0 doc3 doc6 doc7 doc8
 aer 8 doc0 doc1 doc3 doc4 doc5 doc6 doc7 doc8
@@ -805,11 +823,11 @@
 echipa 6 doc0 doc3 doc5 doc6 doc7 doc8
 ecou 7 doc0 doc3 doc4 doc5 doc6 doc7 doc8
 efort 6 doc0 doc3 doc5 doc6 doc7 doc8
-emotie 6 doc0 doc3 doc4 doc5 doc7 doc8
+emotie 5 doc3 doc4 doc5 doc7 doc8
 explorare 8 doc0 doc1 doc3 doc4 doc5 doc6 doc7 doc8
 faleza 7 doc0 doc1 doc3 doc4 doc5 doc7 doc8
 floare 6 doc0 doc3 doc5 doc6 doc7 doc8
-fluviu 7 doc0 doc3 doc4 doc5 doc6 doc7 doc8
+fluviu 6 doc3 doc4 doc5 doc6 doc7 doc8
 foc 7 doc0 doc3 doc4 doc5 doc6 doc7 doc8
 formatiuni 6 doc0 doc1 doc3 doc5 doc7 doc8
 frica 5 doc0 doc4 doc6 doc7 doc8
@@ -839,7 +857,7 @@
 masini 7 doc0 doc1 doc3 doc4 doc5 doc7 doc8
 mister 6 doc0 doc3 doc5 doc6 doc7 doc8
 momeala 6 doc0 doc3 doc4 doc5 doc7 doc8
-motor 4 doc0 doc3 doc5 doc7
+motor 3 doc3 doc5 doc7
 munte 6 doc0 doc3 doc4 doc6 doc7 doc8
 muzica 6 doc0 doc3 doc4 doc5 doc7 doc8
 natura 6 doc1 doc3 doc5 doc6 doc7 doc8
@@ -850,7 +868,7 @@
 oras 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
 orientare 6 doc0 doc3 doc4 doc5 doc7 doc8
 otel 5 doc0 doc1 doc3 doc5 doc7
-padure 7 doc0 doc1 doc3 doc4 doc5 doc6 doc7
+padure 6 doc0 doc1 doc3 doc4 doc5 doc7
 palmieri 7 doc0 doc3 doc4 doc5 doc6 doc7 doc8
 pamant 7 doc0 doc1 doc3 doc4 doc5 doc7 doc8
 parcare 5 doc0 doc1 doc3 doc4 doc7
@@ -863,7 +881,7 @@
 pescuit 8 doc0 doc1 doc3 doc4 doc5 doc6 doc7 doc8
 peste 6 doc0 doc1 doc3 doc5 doc7 doc8
 pestera 5 doc0 doc3 doc6 doc7 doc8
-picaturi 6 doc0 doc3 doc5 doc6 doc7 doc8
+picaturi 5 doc0 doc5 doc6 doc7 doc8
 plaja 7 doc0 doc3 doc4 doc5 doc6 doc7 doc8
 plasa 4 doc0 doc4 doc5 doc7
 ploaie 6 doc0 doc3 doc5 doc6 doc7 doc8
@@ -871,18 +889,18 @@
 poiana 6 doc0 doc1 doc4 doc5 doc7 doc8
 port 5 doc0 doc3 doc4 doc7 doc8
 povesti 5 doc0 doc3 doc5 doc7 doc8
-poza 4 doc0 doc1 doc7 doc8
+poza 3 doc1 doc7 doc8
 poze 6 doc0 doc3 doc4 doc5 doc7 doc8
 prapastie 5 doc0 doc3 doc6 doc7 doc8
 prieteni 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
-rasarit 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
+rasarit 6 doc0 doc1 doc3 doc5 doc6 doc8
 rece 5 doc0 doc1 doc3 doc5 doc7
 reflectie 6 doc0 doc3 doc5 doc6 doc7 doc8
 relaxare 6 doc0 doc3 doc5 doc6 doc7 doc8
 rucsac 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
 salcie 5 doc0 doc3 doc5 doc7 doc8
 sanie 5 doc0 doc3 doc4 doc7 doc8
-sare 6 doc0 doc1 doc5 doc6 doc7 doc8
+sare 5 doc0 doc1 doc6 doc7 doc8
 schi 8 doc0 doc1 doc3 doc4 doc5 doc6 doc7 doc8
 scoici 5 doc0 doc5 doc6 doc7 doc8
 semineu 5 doc0 doc3 doc4 doc5 doc7
@@ -896,7 +914,7 @@
 stuf 8 doc0 doc1 doc3 doc4 doc5 doc6 doc7 doc8
 tacere 6 doc0 doc1 doc3 doc5 doc6 doc7
 tehnologie 8 doc0 doc1 doc3 doc4 doc5 doc6 doc7 doc8
-timp 5 doc0 doc3 doc5 doc6 doc7
+timp 4 doc3 doc5 doc6 doc7
 transport 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
 traseu 4 doc0 doc3 doc5 doc7
 trezire 5 doc0 doc1 doc3 doc5 doc7
@@ -915,8 +933,11 @@
 vis 5 doc0 doc4 doc5 doc7 doc8
 zapada 6 doc0 doc3 doc5 doc6 doc7 doc8
 zgomot 6 doc0 doc1 doc5 doc6 doc7 doc8
+5 doc0 doc4 doc5 doc7 doc8
 OK
 OK
+NOT FOUND
+NOT FOUND
 adancime 5 doc0 doc5 doc6 doc7 doc8
 adrenalina 5 doc0 doc3 doc6 doc7 doc8
 aer 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
@@ -957,11 +978,11 @@
 echipa 6 doc0 doc3 doc5 doc6 doc7 doc8
 ecou 6 doc0 doc3 doc5 doc6 doc7 doc8
 efort 6 doc0 doc3 doc5 doc6 doc7 doc8
-emotie 5 doc0 doc3 doc5 doc7 doc8
+emotie 4 doc3 doc5 doc7 doc8
 explorare 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
 faleza 6 doc0 doc1 doc3 doc5 doc7 doc8
 floare 6 doc0 doc3 doc5 doc6 doc7 doc8
-fluviu 6 doc0 doc3 doc5 doc6 doc7 doc8
+fluviu 5 doc3 doc5 doc6 doc7 doc8
 foc 6 doc0 doc3 doc5 doc6 doc7 doc8
 formatiuni 6 doc0 doc1 doc3 doc5 doc7 doc8
 frica 4 doc0 doc6 doc7 doc8
@@ -991,7 +1012,7 @@
 masini 6 doc0 doc1 doc3 doc5 doc7 doc8
 mister 6 doc0 doc3 doc5 doc6 doc7 doc8
 momeala 5 doc0 doc3 doc5 doc7 doc8
-motor 4 doc0 doc3 doc5 doc7
+motor 3 doc3 doc5 doc7
 munte 5 doc0 doc3 doc6 doc7 doc8
 muzica 5 doc0 doc3 doc5 doc7 doc8
 natura 6 doc1 doc3 doc5 doc6 doc7 doc8
@@ -1002,7 +1023,7 @@
 oras 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
 orientare 5 doc0 doc3 doc5 doc7 doc8
 otel 5 doc0 doc1 doc3 doc5 doc7
-padure 6 doc0 doc1 doc3 doc5 doc6 doc7
+padure 5 doc0 doc1 doc3 doc5 doc7
 palmieri 6 doc0 doc3 doc5 doc6 doc7 doc8
 pamant 6 doc0 doc1 doc3 doc5 doc7 doc8
 parcare 4 doc0 doc1 doc3 doc7
@@ -1015,7 +1036,7 @@
 pescuit 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
 peste 6 doc0 doc1 doc3 doc5 doc7 doc8
 pestera 5 doc0 doc3 doc6 doc7 doc8
-picaturi 6 doc0 doc3 doc5 doc6 doc7 doc8
+picaturi 5 doc0 doc5 doc6 doc7 doc8
 plaja 6 doc0 doc3 doc5 doc6 doc7 doc8
 plasa 3 doc0 doc5 doc7
 ploaie 6 doc0 doc3 doc5 doc6 doc7 doc8
@@ -1023,18 +1044,18 @@
 poiana 5 doc0 doc1 doc5 doc7 doc8
 port 4 doc0 doc3 doc7 doc8
 povesti 5 doc0 doc3 doc5 doc7 doc8
-poza 4 doc0 doc1 doc7 doc8
+poza 3 doc1 doc7 doc8
 poze 5 doc0 doc3 doc5 doc7 doc8
 prapastie 5 doc0 doc3 doc6 doc7 doc8
 prieteni 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
-rasarit 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
+rasarit 6 doc0 doc1 doc3 doc5 doc6 doc8
 rece 5 doc0 doc1 doc3 doc5 doc7
 reflectie 6 doc0 doc3 doc5 doc6 doc7 doc8
 relaxare 6 doc0 doc3 doc5 doc6 doc7 doc8
 rucsac 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
 salcie 5 doc0 doc3 doc5 doc7 doc8
 sanie 4 doc0 doc3 doc7 doc8
-sare 6 doc0 doc1 doc5 doc6 doc7 doc8
+sare 5 doc0 doc1 doc6 doc7 doc8
 schi 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
 scoici 5 doc0 doc5 doc6 doc7 doc8
 semineu 4 doc0 doc3 doc5 doc7
@@ -1048,7 +1069,7 @@
 stuf 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
 tacere 6 doc0 doc1 doc3 doc5 doc6 doc7
 tehnologie 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
-timp 5 doc0 doc3 doc5 doc6 doc7
+timp 4 doc3 doc5 doc6 doc7
 transport 7 doc0 doc1 doc3 doc5 doc6 doc7 doc8
 traseu 4 doc0 doc3 doc5 doc7
 trezire 5 doc0 doc1 doc3 doc5 doc7
@@ -1067,9 +1088,13 @@
 vis 4 doc0 doc5 doc7 doc8
 zapada 6 doc0 doc3 doc5 doc6 doc7 doc8
 zgomot 6 doc0 doc1 doc5 doc6 doc7 doc8
+3 doc0 doc5 doc7
 NOT FOUND
 OK
 NOT FOUND
+5 doc0 doc3 doc5 doc7 doc8
+NOT FOUND
+OK
 adancime 5 doc0 doc5 doc6 doc7 doc8
 adrenalina 5 doc0 doc3 doc6 doc7 doc8
 aer 6 doc0 doc3 doc5 doc6 doc7 doc8
@@ -1110,11 +1135,11 @@
 echipa 6 doc0 doc3 doc5 doc6 doc7 doc8
 ecou 6 doc0 doc3 doc5 doc6 doc7 doc8
 efort 6 doc0 doc3 doc5 doc6 doc7 doc8
-emotie 5 doc0 doc3 doc5 doc7 doc8
+emotie 4 doc3 doc5 doc7 doc8
 explorare 6 doc0 doc3 doc5 doc6 doc7 doc8
 faleza 5 doc0 doc3 doc5 doc7 doc8
 floare 6 doc0 doc3 doc5 doc6 doc7 doc8
-fluviu 6 doc0 doc3 doc5 doc6 doc7 doc8
+fluviu 5 doc3 doc5 doc6 doc7 doc8
 foc 6 doc0 doc3 doc5 doc6 doc7 doc8
 formatiuni 5 doc0 doc3 doc5 doc7 doc8
 frica 4 doc0 doc6 doc7 doc8
@@ -1144,7 +1169,7 @@
 masini 5 doc0 doc3 doc5 doc7 doc8
 mister 6 doc0 doc3 doc5 doc6 doc7 doc8
 momeala 5 doc0 doc3 doc5 doc7 doc8
-motor 4 doc0 doc3 doc5 doc7
+motor 3 doc3 doc5 doc7
 munte 5 doc0 doc3 doc6 doc7 doc8
 muzica 5 doc0 doc3 doc5 doc7 doc8
 natura 5 doc3 doc5 doc6 doc7 doc8
@@ -1155,7 +1180,7 @@
 oras 6 doc0 doc3 doc5 doc6 doc7 doc8
 orientare 5 doc0 doc3 doc5 doc7 doc8
 otel 4 doc0 doc3 doc5 doc7
-padure 5 doc0 doc3 doc5 doc6 doc7
+padure 4 doc0 doc3 doc5 doc7
 palmieri 6 doc0 doc3 doc5 doc6 doc7 doc8
 pamant 5 doc0 doc3 doc5 doc7 doc8
 parcare 3 doc0 doc3 doc7
@@ -1168,7 +1193,7 @@
 pescuit 6 doc0 doc3 doc5 doc6 doc7 doc8
 peste 5 doc0 doc3 doc5 doc7 doc8
 pestera 5 doc0 doc3 doc6 doc7 doc8
-picaturi 6 doc0 doc3 doc5 doc6 doc7 doc8
+picaturi 5 doc0 doc5 doc6 doc7 doc8
 plaja 6 doc0 doc3 doc5 doc6 doc7 doc8
 plasa 3 doc0 doc5 doc7
 ploaie 6 doc0 doc3 doc5 doc6 doc7 doc8
@@ -1176,18 +1201,18 @@
 poiana 4 doc0 doc5 doc7 doc8
 port 4 doc0 doc3 doc7 doc8
 povesti 5 doc0 doc3 doc5 doc7 doc8
-poza 3 doc0 doc7 doc8
+poza 2 doc7 doc8
 poze 5 doc0 doc3 doc5 doc7 doc8
 prapastie 5 doc0 doc3 doc6 doc7 doc8
 prieteni 6 doc0 doc3 doc5 doc6 doc7 doc8
-rasarit 6 doc0 doc3 doc5 doc6 doc7 doc8
+rasarit 5 doc0 doc3 doc5 doc6 doc8
 rece 4 doc0 doc3 doc5 doc7
 reflectie 6 doc0 doc3 doc5 doc6 doc7 doc8
 relaxare 6 doc0 doc3 doc5 doc6 doc7 doc8
 rucsac 6 doc0 doc3 doc5 doc6 doc7 doc8
 salcie 5 doc0 doc3 doc5 doc7 doc8
 sanie 4 doc0 doc3 doc7 doc8
-sare 5 doc0 doc5 doc6 doc7 doc8
+sare 4 doc0 doc6 doc7 doc8
 schi 6 doc0 doc3 doc5 doc6 doc7 doc8
 scoici 5 doc0 doc5 doc6 doc7 doc8
 semineu 4 doc0 doc3 doc5 doc7
@@ -1201,7 +1226,7 @@
 stuf 6 doc0 doc3 doc5 doc6 doc7 doc8
 tacere 5 doc0 doc3 doc5 doc6 doc7
 tehnologie 6 doc0 doc3 doc5 doc6 doc7 doc8
-timp 5 doc0 doc3 doc5 doc6 doc7
+timp 4 doc3 doc5 doc6 doc7
 transport 6 doc0 doc3 doc5 doc6 doc7 doc8
 traseu 4 doc0 doc3 doc5 doc7
 trezire 4 doc0 doc3 doc5 doc7
@@ -1210,7 +1235,7 @@
 umbrela 3 doc0 doc7 doc8
 undita 6 doc0 doc3 doc5 doc6 doc7 doc8
 urs 6 doc0 doc3 doc5 doc6 doc7 doc8
-valuri 5 doc0 doc3 doc5 doc7 doc8
+valuri 4 doc0 doc5 doc7 doc8
 vant 3 doc0 doc5 doc8
 vapor 6 doc0 doc3 doc5 doc6 doc7 doc8
 vasle 5 doc0 doc3 doc6 doc7 doc8
@@ -1220,6 +1245,7 @@
 vis 4 doc0 doc5 doc7 doc8
 zapada 6 doc0 doc3 doc5 doc6 doc7 doc8
 zgomot 5 doc0 doc5 doc6 doc7 doc8
+NOT FOUND
 adancime 5 doc0 doc5 doc6 doc7 doc8
 adrenalina 5 doc0 doc3 doc6 doc7 doc8
 aer 6 doc0 doc3 doc5 doc6 doc7 doc8
@@ -1260,11 +1286,11 @@
 echipa 6 doc0 doc3 doc5 doc6 doc7 doc8
 ecou 6 doc0 doc3 doc5 doc6 doc7 doc8
 efort 6 doc0 doc3 doc5 doc6 doc7 doc8
-emotie 5 doc0 doc3 doc5 doc7 doc8
+emotie 4 doc3 doc5 doc7 doc8
 explorare 6 doc0 doc3 doc5 doc6 doc7 doc8
 faleza 5 doc0 doc3 doc5 doc7 doc8
 floare 6 doc0 doc3 doc5 doc6 doc7 doc8
-fluviu 6 doc0 doc3 doc5 doc6 doc7 doc8
+fluviu 5 doc3 doc5 doc6 doc7 doc8
 foc 6 doc0 doc3 doc5 doc6 doc7 doc8
 formatiuni 5 doc0 doc3 doc5 doc7 doc8
 frica 4 doc0 doc6 doc7 doc8
@@ -1294,7 +1320,7 @@
 masini 5 doc0 doc3 doc5 doc7 doc8
 mister 6 doc0 doc3 doc5 doc6 doc7 doc8
 momeala 5 doc0 doc3 doc5 doc7 doc8
-motor 4 doc0 doc3 doc5 doc7
+motor 3 doc3 doc5 doc7
 munte 5 doc0 doc3 doc6 doc7 doc8
 muzica 5 doc0 doc3 doc5 doc7 doc8
 natura 5 doc3 doc5 doc6 doc7 doc8
@@ -1305,7 +1331,7 @@
 oras 6 doc0 doc3 doc5 doc6 doc7 doc8
 orientare 5 doc0 doc3 doc5 doc7 doc8
 otel 4 doc0 doc3 doc5 doc7
-padure 5 doc0 doc3 doc5 doc6 doc7
+padure 4 doc0 doc3 doc5 doc7
 palmieri 6 doc0 doc3 doc5 doc6 doc7 doc8
 pamant 5 doc0 doc3 doc5 doc7 doc8
 parcare 3 doc0 doc3 doc7
@@ -1318,7 +1344,7 @@
 pescuit 6 doc0 doc3 doc5 doc6 doc7 doc8
 peste 5 doc0 doc3 doc5 doc7 doc8
 pestera 5 doc0 doc3 doc6 doc7 doc8
-picaturi 6 doc0 doc3 doc5 doc6 doc7 doc8
+picaturi 5 doc0 doc5 doc6 doc7 doc8
 plaja 6 doc0 doc3 doc5 doc6 doc7 doc8
 plasa 3 doc0 doc5 doc7
 ploaie 6 doc0 doc3 doc5 doc6 doc7 doc8
@@ -1326,18 +1352,18 @@
 poiana 4 doc0 doc5 doc7 doc8
 port 4 doc0 doc3 doc7 doc8
 povesti 5 doc0 doc3 doc5 doc7 doc8
-poza 3 doc0 doc7 doc8
+poza 2 doc7 doc8
 poze 5 doc0 doc3 doc5 doc7 doc8
 prapastie 5 doc0 doc3 doc6 doc7 doc8
 prieteni 6 doc0 doc3 doc5 doc6 doc7 doc8
-rasarit 6 doc0 doc3 doc5 doc6 doc7 doc8
+rasarit 5 doc0 doc3 doc5 doc6 doc8
 rece 4 doc0 doc3 doc5 doc7
 reflectie 6 doc0 doc3 doc5 doc6 doc7 doc8
 relaxare 6 doc0 doc3 doc5 doc6 doc7 doc8
 rucsac 6 doc0 doc3 doc5 doc6 doc7 doc8
 salcie 5 doc0 doc3 doc5 doc7 doc8
 sanie 4 doc0 doc3 doc7 doc8
-sare 5 doc0 doc5 doc6 doc7 doc8
+sare 4 doc0 doc6 doc7 doc8
 schi 6 doc0 doc3 doc5 doc6 doc7 doc8
 scoici 5 doc0 doc5 doc6 doc7 doc8
 semineu 4 doc0 doc3 doc5 doc7
@@ -1351,7 +1377,7 @@
 stuf 6 doc0 doc3 doc5 doc6 doc7 doc8
 tacere 5 doc0 doc3 doc5 doc6 doc7
 tehnologie 6 doc0 doc3 doc5 doc6 doc7 doc8
-timp 5 doc0 doc3 doc5 doc6 doc7
+timp 4 doc3 doc5 doc6 doc7
 transport 6 doc0 doc3 doc5 doc6 doc7 doc8
 traseu 4 doc0 doc3 doc5 doc7
 trezire 4 doc0 doc3 doc5 doc7
@@ -1360,7 +1386,7 @@
 umbrela 3 doc0 doc7 doc8
 undita 6 doc0 doc3 doc5 doc6 doc7 doc8
 urs 6 doc0 doc3 doc5 doc6 doc7 doc8
-valuri 5 doc0 doc3 doc5 doc7 doc8
+valuri 4 doc0 doc5 doc7 doc8
 vant 3 doc0 doc5 doc8
 vapor 6 doc0 doc3 doc5 doc6 doc7 doc8
 vasle 5 doc0 doc3 doc6 doc7 doc8
@@ -1370,6 +1396,946 @@
 vis 4 doc0 doc5 doc7 doc8
 zapada 6 doc0 doc3 doc5 doc6 doc7 doc8
 zgomot 5 doc0 doc5 doc6 doc7 doc8
+4 doc3 doc5 doc7 doc8
+OK
+4 doc0 doc3 doc5 doc7
+OK
+OK
+NOT FOUND
+7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+OK
+OK
+adancime 6 doc0 doc5 doc6 doc7 doc8 doc9
+adrenalina 6 doc0 doc3 doc6 doc7 doc8 doc9
+aer 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+afaceri 6 doc0 doc3 doc6 doc7 doc8 doc9
+aglomeratie 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+amintiri 6 doc3 doc5 doc6 doc7 doc8 doc9
+amurg 6 doc0 doc3 doc5 doc7 doc8 doc9
+ancora 6 doc0 doc3 doc5 doc7 doc8 doc9
+apa 5 doc0 doc3 doc6 doc7 doc8
+aventura 6 doc3 doc5 doc6 doc7 doc8 doc9
+bani 5 doc0 doc3 doc6 doc7 doc9
+barca 4 doc0 doc3 doc5 doc7
+barcagiu 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+baterii 6 doc0 doc3 doc6 doc7 doc8 doc9
+beton 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+bocanci 3 doc3 doc7 doc9
+brad 5 doc0 doc6 doc7 doc8 doc9
+busola 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+cabana 5 doc0 doc3 doc7 doc8 doc9
+cabanier 5 doc0 doc3 doc7 doc8 doc9
+cald 5 doc0 doc3 doc6 doc7 doc9
+campie 5 doc0 doc3 doc5 doc7 doc9
+canale 5 doc3 doc5 doc7 doc8 doc9
+ceai 6 doc0 doc3 doc5 doc7 doc8 doc9
+ceata 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+comert 4 doc0 doc3 doc7 doc9
+cort 3 doc0 doc7 doc9
+cristal 6 doc0 doc3 doc5 doc7 doc8 doc9
+curaj 6 doc0 doc3 doc6 doc7 doc8 doc9
+curat 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+curent 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+dans 6 doc0 doc3 doc5 doc6 doc7 doc9
+delta 3 doc0 doc6 doc7
+dezvoltare 5 doc0 doc3 doc5 doc7 doc8
+dig 6 doc0 doc3 doc5 doc7 doc8 doc9
+distractie 6 doc0 doc3 doc6 doc7 doc8 doc9
+drum 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+echipa 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+ecou 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+efort 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+emotie 5 doc3 doc5 doc7 doc8 doc9
+explorare 6 doc0 doc3 doc6 doc7 doc8 doc9
+faleza 6 doc0 doc3 doc5 doc7 doc8 doc9
+floare 6 doc0 doc3 doc5 doc6 doc7 doc8
+fluviu 6 doc3 doc5 doc6 doc7 doc8 doc9
+foc 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+formatiuni 6 doc0 doc3 doc5 doc7 doc8 doc9
+frica 5 doc0 doc6 doc7 doc8 doc9
+frig 6 doc0 doc3 doc5 doc6 doc7 doc8
+gheata 5 doc0 doc6 doc7 doc8 doc9
+harta 5 doc0 doc5 doc6 doc7 doc9
+iarna 5 doc0 doc6 doc7 doc8 doc9
+indigo 6 doc0 doc3 doc6 doc7 doc8 doc9
+informatii 6 doc0 doc3 doc5 doc7 doc8 doc9
+inot 5 doc0 doc3 doc5 doc7 doc9
+insula 4 doc6 doc7 doc8 doc9
+intuneric 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+istorie 5 doc0 doc3 doc5 doc7 doc8
+izvor 5 doc0 doc3 doc5 doc7 doc9
+lanterna 4 doc0 doc7 doc8 doc9
+lemne 6 doc0 doc3 doc5 doc6 doc7 doc9
+liniste 5 doc0 doc3 doc7 doc8 doc9
+lume 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+lumina 6 doc3 doc5 doc6 doc7 doc8 doc9
+lumini 5 doc0 doc3 doc5 doc7 doc9
+lup 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+mal 5 doc0 doc3 doc6 doc7 doc9
+mare 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+marfa 4 doc3 doc6 doc7 doc9
+marinari 3 doc7 doc8 doc9
+masina 5 doc3 doc5 doc6 doc7 doc9
+masini 6 doc0 doc3 doc5 doc7 doc8 doc9
+mister 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+momeala 6 doc0 doc3 doc5 doc7 doc8 doc9
+motor 4 doc3 doc5 doc7 doc9
+munte 6 doc0 doc3 doc6 doc7 doc8 doc9
+muzica 6 doc0 doc3 doc5 doc7 doc8 doc9
+natura 6 doc3 doc5 doc6 doc7 doc8 doc9
+nisip 6 doc0 doc3 doc6 doc7 doc8 doc9
+noapte 6 doc3 doc5 doc6 doc7 doc8 doc9
+nor 4 doc0 doc5 doc6 doc7
+nuca 6 doc0 doc3 doc5 doc6 doc7 doc9
+oras 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+orientare 6 doc0 doc3 doc5 doc7 doc8 doc9
+otel 4 doc0 doc3 doc5 doc7
+padure 5 doc0 doc3 doc5 doc7 doc9
+palmieri 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+pamant 6 doc0 doc3 doc5 doc7 doc8 doc9
+parcare 4 doc0 doc3 doc7 doc9
+pasari 6 doc0 doc3 doc5 doc7 doc8 doc9
+patine 5 doc0 doc5 doc7 doc8 doc9
+patura 6 doc0 doc3 doc5 doc7 doc8 doc9
+peisaj 6 doc0 doc3 doc5 doc6 doc7 doc9
+pelican 6 doc0 doc5 doc6 doc7 doc8 doc9
+pescarusi 5 doc0 doc3 doc7 doc8 doc9
+pescuit 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+peste 6 doc0 doc3 doc5 doc7 doc8 doc9
+pestera 6 doc0 doc3 doc6 doc7 doc8 doc9
+picaturi 6 doc0 doc5 doc6 doc7 doc8 doc9
+plaja 6 doc0 doc3 doc5 doc6 doc7 doc8
+plasa 4 doc0 doc5 doc7 doc9
+ploaie 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+pod 5 doc0 doc5 doc7 doc8 doc9
+poiana 5 doc0 doc5 doc7 doc8 doc9
+port 5 doc0 doc3 doc7 doc8 doc9
+povesti 6 doc0 doc3 doc5 doc7 doc8 doc9
+poza 2 doc7 doc8
+poze 6 doc0 doc3 doc5 doc7 doc8 doc9
+prapastie 6 doc0 doc3 doc6 doc7 doc8 doc9
+prieteni 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+rasarit 6 doc0 doc3 doc5 doc6 doc8 doc9
+rece 5 doc0 doc3 doc5 doc7 doc9
+reflectie 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+relaxare 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+rucsac 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+salcie 6 doc0 doc3 doc5 doc7 doc8 doc9
+sanie 5 doc0 doc3 doc7 doc8 doc9
+sare 5 doc0 doc6 doc7 doc8 doc9
+schi 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+scoici 6 doc0 doc5 doc6 doc7 doc8 doc9
+semineu 5 doc0 doc3 doc5 doc7 doc9
+sezlong 6 doc0 doc5 doc6 doc7 doc8 doc9
+soare 6 doc0 doc3 doc5 doc7 doc8 doc9
+somn 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+sport 5 doc0 doc5 doc7 doc8 doc9
+stanca 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+stele 5 doc0 doc3 doc7 doc8 doc9
+sticla 4 doc3 doc6 doc7 doc9
+stuf 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+tacere 6 doc0 doc3 doc5 doc6 doc7 doc9
+tehnologie 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+timp 5 doc3 doc5 doc6 doc7 doc9
+transport 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+traseu 5 doc0 doc3 doc5 doc7 doc9
+trezire 5 doc0 doc3 doc5 doc7 doc9
+turist 6 doc0 doc3 doc5 doc6 doc7 doc9
+umbra 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+umbrela 4 doc0 doc7 doc8 doc9
+undita 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+urs 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+valuri 5 doc0 doc5 doc7 doc8 doc9
+vant 3 doc0 doc5 doc8
+vapor 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+vasle 6 doc0 doc3 doc6 doc7 doc8 doc9
+viitor 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+vin 6 doc0 doc3 doc5 doc7 doc8 doc9
+violet 5 doc0 doc6 doc7 doc8 doc9
+vis 5 doc0 doc5 doc7 doc8 doc9
+zapada 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+zgomot 6 doc0 doc5 doc6 doc7 doc8 doc9
+5 doc0 doc5 doc7 doc8 doc9
+OK
+OK
+OK
+7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+OK
+OK
+OK
+6 doc0 doc3 doc5 doc7 doc8 doc9
+NOT FOUND
+8 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc9
+OK
+NOT FOUND
+adancime 7 doc0 doc11 doc5 doc6 doc7 doc8 doc9
+adrenalina 8 doc0 doc10 doc11 doc3 doc6 doc7 doc8 doc9
+aer 8 doc0 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+afaceri 6 doc0 doc3 doc6 doc7 doc8 doc9
+aglomeratie 8 doc0 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+amintiri 7 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+amurg 6 doc0 doc3 doc5 doc7 doc8 doc9
+ancora 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+apa 7 doc0 doc10 doc11 doc3 doc6 doc7 doc8
+aventura 8 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+bani 7 doc0 doc10 doc11 doc3 doc6 doc7 doc9
+barca 6 doc0 doc10 doc11 doc3 doc5 doc7
+barcagiu 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+baterii 8 doc0 doc10 doc11 doc3 doc6 doc7 doc8 doc9
+beton 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+bocanci 4 doc10 doc3 doc7 doc9
+brad 7 doc0 doc10 doc11 doc6 doc7 doc8 doc9
+busola 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+cabana 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+cabanier 6 doc0 doc11 doc3 doc7 doc8 doc9
+cald 7 doc0 doc10 doc11 doc3 doc6 doc7 doc9
+campie 6 doc0 doc10 doc3 doc5 doc7 doc9
+canale 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+ceai 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+ceata 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+comert 6 doc0 doc10 doc11 doc3 doc7 doc9
+cort 4 doc0 doc10 doc7 doc9
+cristal 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+curaj 8 doc0 doc10 doc11 doc3 doc6 doc7 doc8 doc9
+curat 8 doc0 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+curent 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+dans 8 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc9
+delta 5 doc0 doc10 doc11 doc6 doc7
+dezvoltare 6 doc0 doc10 doc3 doc5 doc7 doc8
+dig 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+distractie 7 doc0 doc11 doc3 doc6 doc7 doc8 doc9
+drum 8 doc0 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+echipa 8 doc0 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+ecou 8 doc0 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+efort 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+emotie 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+explorare 8 doc0 doc10 doc11 doc3 doc6 doc7 doc8 doc9
+faleza 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+floare 7 doc0 doc11 doc3 doc5 doc6 doc7 doc8
+fluviu 7 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+foc 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+formatiuni 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+frica 7 doc0 doc10 doc11 doc6 doc7 doc8 doc9
+frig 7 doc0 doc10 doc3 doc5 doc6 doc7 doc8
+gheata 7 doc0 doc10 doc11 doc6 doc7 doc8 doc9
+harta 7 doc0 doc10 doc11 doc5 doc6 doc7 doc9
+iarna 6 doc0 doc10 doc6 doc7 doc8 doc9
+indigo 7 doc0 doc11 doc3 doc6 doc7 doc8 doc9
+informatii 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+inot 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+insula 5 doc10 doc6 doc7 doc8 doc9
+intuneric 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+istorie 6 doc0 doc10 doc3 doc5 doc7 doc8
+izvor 5 doc0 doc3 doc5 doc7 doc9
+lanterna 5 doc0 doc10 doc7 doc8 doc9
+lemne 8 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc9
+liniste 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+lume 8 doc0 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+lumina 8 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+lumini 6 doc0 doc10 doc11 doc5 doc7 doc9
+lup 8 doc0 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+mal 7 doc0 doc10 doc11 doc3 doc6 doc7 doc9
+mare 8 doc0 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+marfa 5 doc10 doc3 doc6 doc7 doc9
+marinari 5 doc10 doc11 doc7 doc8 doc9
+masina 6 doc10 doc3 doc5 doc6 doc7 doc9
+masini 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+mister 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+momeala 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+motor 6 doc10 doc11 doc3 doc5 doc7 doc9
+munte 7 doc0 doc10 doc3 doc6 doc7 doc8 doc9
+muzica 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+natura 8 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+nisip 8 doc0 doc10 doc11 doc3 doc6 doc7 doc8 doc9
+noapte 8 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+nor 6 doc0 doc10 doc11 doc5 doc6 doc7
+nuca 8 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc9
+oras 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+orientare 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+otel 6 doc0 doc10 doc11 doc3 doc5 doc7
+padure 6 doc0 doc10 doc11 doc3 doc7 doc9
+palmieri 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+pamant 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+parcare 5 doc0 doc10 doc3 doc7 doc9
+pasari 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+patine 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+patura 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+peisaj 8 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc9
+pelican 8 doc0 doc10 doc11 doc5 doc6 doc7 doc8 doc9
+pescarusi 6 doc0 doc10 doc3 doc7 doc8 doc9
+pescuit 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+peste 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+pestera 8 doc0 doc10 doc11 doc3 doc6 doc7 doc8 doc9
+picaturi 8 doc0 doc10 doc11 doc5 doc6 doc7 doc8 doc9
+plaja 7 doc0 doc10 doc3 doc5 doc6 doc7 doc8
+plasa 5 doc0 doc11 doc5 doc7 doc9
+ploaie 8 doc0 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+pod 6 doc0 doc10 doc5 doc7 doc8 doc9
+poiana 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+port 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+povesti 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+poza 4 doc10 doc11 doc7 doc8
+poze 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+prapastie 7 doc0 doc10 doc3 doc6 doc7 doc8 doc9
+prieteni 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+rasarit 8 doc0 doc10 doc11 doc3 doc5 doc6 doc8 doc9
+rece 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+reflectie 8 doc0 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+relaxare 8 doc0 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+rucsac 8 doc0 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+salcie 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+sanie 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+sare 7 doc0 doc10 doc11 doc6 doc7 doc8 doc9
+schi 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+scoici 8 doc0 doc10 doc11 doc5 doc6 doc7 doc8 doc9
+semineu 6 doc0 doc10 doc3 doc5 doc7 doc9
+sezlong 7 doc0 doc10 doc5 doc6 doc7 doc8 doc9
+soare 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+somn 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+sport 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+stanca 8 doc0 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+stele 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+sticla 6 doc10 doc11 doc3 doc6 doc7 doc9
+stuf 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+tacere 8 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc9
+tehnologie 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+timp 7 doc10 doc11 doc3 doc5 doc6 doc7 doc9
+transport 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+traseu 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+trezire 6 doc0 doc10 doc3 doc5 doc7 doc9
+turist 8 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc9
+umbra 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+umbrela 6 doc0 doc10 doc11 doc7 doc8 doc9
+undita 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+urs 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+valuri 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+vant 4 doc0 doc10 doc5 doc8
+vapor 8 doc0 doc10 doc3 doc5 doc6 doc7 doc8 doc9
+vasle 8 doc0 doc10 doc11 doc3 doc6 doc7 doc8 doc9
+viitor 9 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc8 doc9
+vin 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+violet 7 doc0 doc10 doc11 doc6 doc7 doc8 doc9
+vis 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+zapada 7 doc0 doc3 doc5 doc6 doc7 doc8 doc9
+zgomot 7 doc0 doc11 doc5 doc6 doc7 doc8 doc9
+8 doc0 doc10 doc11 doc3 doc5 doc6 doc7 doc9
+OK
+OK
+OK
+8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+OK
+NOT FOUND
+NOT FOUND
+7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+OK
+8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+OK
+OK
+adancime 6 doc0 doc11 doc5 doc7 doc8 doc9
+adrenalina 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+aer 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+afaceri 5 doc0 doc3 doc7 doc8 doc9
+aglomeratie 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+amintiri 6 doc10 doc3 doc5 doc7 doc8 doc9
+amurg 6 doc0 doc3 doc5 doc7 doc8 doc9
+ancora 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+apa 6 doc0 doc10 doc11 doc3 doc7 doc8
+aventura 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+bani 6 doc0 doc10 doc11 doc3 doc7 doc9
+barca 6 doc0 doc10 doc11 doc3 doc5 doc7
+barcagiu 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+baterii 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+beton 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+bocanci 4 doc10 doc3 doc7 doc9
+brad 6 doc0 doc10 doc11 doc7 doc8 doc9
+busola 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+cabana 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+cabanier 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+cald 6 doc0 doc10 doc11 doc3 doc7 doc9
+campie 6 doc0 doc10 doc3 doc5 doc7 doc9
+canale 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+ceai 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+ceata 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+comert 6 doc0 doc10 doc11 doc3 doc7 doc9
+cort 4 doc0 doc10 doc7 doc9
+cristal 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+curaj 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+curat 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+curent 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+dans 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+delta 4 doc0 doc10 doc11 doc7
+dezvoltare 6 doc0 doc10 doc3 doc5 doc7 doc8
+dig 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+distractie 6 doc0 doc11 doc3 doc7 doc8 doc9
+drum 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+echipa 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+ecou 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+efort 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+emotie 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+explorare 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+faleza 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+floare 6 doc0 doc11 doc3 doc5 doc7 doc8
+fluviu 6 doc10 doc3 doc5 doc7 doc8 doc9
+foc 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+formatiuni 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+frica 6 doc0 doc10 doc11 doc7 doc8 doc9
+frig 6 doc0 doc10 doc3 doc5 doc7 doc8
+gheata 6 doc0 doc10 doc11 doc7 doc8 doc9
+harta 6 doc0 doc10 doc11 doc5 doc7 doc9
+iarna 5 doc0 doc10 doc7 doc8 doc9
+indigo 6 doc0 doc11 doc3 doc7 doc8 doc9
+informatii 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+inot 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+insula 4 doc10 doc7 doc8 doc9
+intuneric 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+istorie 6 doc0 doc10 doc3 doc5 doc7 doc8
+izvor 5 doc0 doc3 doc5 doc7 doc9
+lanterna 5 doc0 doc10 doc7 doc8 doc9
+lemne 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+liniste 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+lume 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+lumina 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+lumini 6 doc0 doc10 doc11 doc5 doc7 doc9
+lup 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+mal 6 doc0 doc10 doc11 doc3 doc7 doc9
+mare 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+marfa 4 doc10 doc3 doc7 doc9
+marinari 5 doc10 doc11 doc7 doc8 doc9
+masina 5 doc10 doc3 doc5 doc7 doc9
+masini 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+mister 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+momeala 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+motor 6 doc10 doc11 doc3 doc5 doc7 doc9
+munte 6 doc0 doc10 doc3 doc7 doc8 doc9
+muzica 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+natura 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+nisip 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+noapte 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+nor 5 doc0 doc10 doc11 doc5 doc7
+nuca 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+oras 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+orientare 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+otel 6 doc0 doc10 doc11 doc3 doc5 doc7
+padure 6 doc0 doc10 doc11 doc3 doc7 doc9
+palmieri 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+pamant 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+parcare 5 doc0 doc10 doc3 doc7 doc9
+pasari 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+patine 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+patura 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+peisaj 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+pelican 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+pescarusi 6 doc0 doc10 doc3 doc7 doc8 doc9
+pescuit 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+peste 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+pestera 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+picaturi 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+plaja 6 doc0 doc10 doc3 doc5 doc7 doc8
+plasa 5 doc0 doc11 doc5 doc7 doc9
+ploaie 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+pod 6 doc0 doc10 doc5 doc7 doc8 doc9
+poiana 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+port 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+povesti 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+poza 4 doc10 doc11 doc7 doc8
+poze 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+prapastie 6 doc0 doc10 doc3 doc7 doc8 doc9
+prieteni 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+rasarit 7 doc0 doc10 doc11 doc3 doc5 doc8 doc9
+rece 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+reflectie 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+relaxare 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+rucsac 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+salcie 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+sanie 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+sare 6 doc0 doc10 doc11 doc7 doc8 doc9
+schi 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+scoici 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+semineu 6 doc0 doc10 doc3 doc5 doc7 doc9
+sezlong 6 doc0 doc10 doc5 doc7 doc8 doc9
+soare 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+somn 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+sport 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+stanca 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+stele 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+sticla 5 doc10 doc11 doc3 doc7 doc9
+stuf 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+tacere 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+tehnologie 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+timp 6 doc10 doc11 doc3 doc5 doc7 doc9
+transport 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+traseu 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+trezire 6 doc0 doc10 doc3 doc5 doc7 doc9
+turist 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+umbra 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+umbrela 6 doc0 doc10 doc11 doc7 doc8 doc9
+undita 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+urs 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+valuri 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+vant 4 doc0 doc10 doc5 doc8
+vapor 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+vasle 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+viitor 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+vin 6 doc0 doc10 doc5 doc7 doc8 doc9
+violet 6 doc0 doc10 doc11 doc7 doc8 doc9
+vis 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+zapada 6 doc0 doc3 doc5 doc7 doc8 doc9
+zgomot 6 doc0 doc11 doc5 doc7 doc8 doc9
+OK
+adancime 6 doc0 doc11 doc5 doc7 doc8 doc9
+adrenalina 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+aer 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+afaceri 5 doc0 doc3 doc7 doc8 doc9
+aglomeratie 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+amintiri 6 doc10 doc3 doc5 doc7 doc8 doc9
+amurg 6 doc0 doc3 doc5 doc7 doc8 doc9
+ancora 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+apa 6 doc0 doc10 doc11 doc3 doc7 doc8
+aventura 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+bani 6 doc0 doc10 doc11 doc3 doc7 doc9
+barca 6 doc0 doc10 doc11 doc3 doc5 doc7
+barcagiu 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+baterii 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+beton 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+bocanci 4 doc10 doc3 doc7 doc9
+brad 6 doc0 doc10 doc11 doc7 doc8 doc9
+busola 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+cabana 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+cabanier 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+cald 6 doc0 doc10 doc11 doc3 doc7 doc9
+campie 6 doc0 doc10 doc3 doc5 doc7 doc9
+canale 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+ceai 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+ceata 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+comert 6 doc0 doc10 doc11 doc3 doc7 doc9
+cort 4 doc0 doc10 doc7 doc9
+cristal 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+curaj 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+curat 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+curent 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+dans 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+delta 4 doc0 doc10 doc11 doc7
+dezvoltare 6 doc0 doc10 doc3 doc5 doc7 doc8
+dig 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+distractie 6 doc0 doc11 doc3 doc7 doc8 doc9
+drum 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+echipa 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+ecou 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+efort 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+emotie 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+explorare 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+faleza 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+floare 6 doc0 doc11 doc3 doc5 doc7 doc8
+fluviu 6 doc10 doc3 doc5 doc7 doc8 doc9
+foc 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+formatiuni 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+frica 6 doc0 doc10 doc11 doc7 doc8 doc9
+frig 6 doc0 doc10 doc3 doc5 doc7 doc8
+gheata 6 doc0 doc10 doc11 doc7 doc8 doc9
+harta 6 doc0 doc10 doc11 doc5 doc7 doc9
+iarna 5 doc0 doc10 doc7 doc8 doc9
+indigo 6 doc0 doc11 doc3 doc7 doc8 doc9
+informatii 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+inot 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+insula 4 doc10 doc7 doc8 doc9
+intuneric 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+istorie 6 doc0 doc10 doc3 doc5 doc7 doc8
+izvor 5 doc0 doc3 doc5 doc7 doc9
+lanterna 5 doc0 doc10 doc7 doc8 doc9
+lemne 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+liniste 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+lume 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+lumina 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+lumini 6 doc0 doc10 doc11 doc5 doc7 doc9
+lup 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+mal 6 doc0 doc10 doc11 doc3 doc7 doc9
+mare 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+marfa 4 doc10 doc3 doc7 doc9
+marinari 5 doc10 doc11 doc7 doc8 doc9
+masina 5 doc10 doc3 doc5 doc7 doc9
+masini 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+mister 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+momeala 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+motor 6 doc10 doc11 doc3 doc5 doc7 doc9
+munte 6 doc0 doc10 doc3 doc7 doc8 doc9
+muzica 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+natura 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+nisip 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+noapte 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+nor 5 doc0 doc10 doc11 doc5 doc7
+nuca 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+oras 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+orientare 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+otel 6 doc0 doc10 doc11 doc3 doc5 doc7
+padure 6 doc0 doc10 doc11 doc3 doc7 doc9
+palmieri 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+pamant 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+parcare 5 doc0 doc10 doc3 doc7 doc9
+pasari 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+patine 6 doc0 doc10 doc5 doc7 doc8 doc9
+patura 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+peisaj 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+pelican 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+pescarusi 6 doc0 doc10 doc3 doc7 doc8 doc9
+pescuit 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+peste 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+pestera 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+picaturi 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+plaja 6 doc0 doc10 doc3 doc5 doc7 doc8
+plasa 5 doc0 doc11 doc5 doc7 doc9
+ploaie 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+pod 6 doc0 doc10 doc5 doc7 doc8 doc9
+poiana 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+port 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+povesti 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+poza 4 doc10 doc11 doc7 doc8
+poze 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+prapastie 6 doc0 doc10 doc3 doc7 doc8 doc9
+prieteni 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+rasarit 7 doc0 doc10 doc11 doc3 doc5 doc8 doc9
+rece 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+reflectie 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+relaxare 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+rucsac 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+salcie 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+sanie 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+sare 6 doc0 doc10 doc11 doc7 doc8 doc9
+schi 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+scoici 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+semineu 6 doc0 doc10 doc3 doc5 doc7 doc9
+sezlong 6 doc0 doc10 doc5 doc7 doc8 doc9
+soare 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+somn 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+sport 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+stanca 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+stele 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+sticla 5 doc10 doc11 doc3 doc7 doc9
+stuf 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+tacere 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+tehnologie 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+timp 6 doc10 doc11 doc3 doc5 doc7 doc9
+transport 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+traseu 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+trezire 6 doc0 doc10 doc3 doc5 doc7 doc9
+turist 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+umbra 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+umbrela 6 doc0 doc10 doc11 doc7 doc8 doc9
+undita 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+urs 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+valuri 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+vant 4 doc0 doc10 doc5 doc8
+vapor 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+vasle 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+viitor 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+vin 6 doc0 doc10 doc5 doc7 doc8 doc9
+violet 6 doc0 doc10 doc11 doc7 doc8 doc9
+vis 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+zapada 6 doc0 doc3 doc5 doc7 doc8 doc9
+zgomot 6 doc0 doc11 doc5 doc7 doc8 doc9
+OK
+adancime 6 doc0 doc11 doc5 doc7 doc8 doc9
+adrenalina 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+aer 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+afaceri 5 doc0 doc3 doc7 doc8 doc9
+aglomeratie 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+amintiri 6 doc10 doc3 doc5 doc7 doc8 doc9
+amurg 6 doc0 doc3 doc5 doc7 doc8 doc9
+ancora 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+apa 6 doc0 doc10 doc11 doc3 doc7 doc8
+aventura 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+bani 6 doc0 doc10 doc11 doc3 doc7 doc9
+barca 6 doc0 doc10 doc11 doc3 doc5 doc7
+barcagiu 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+baterii 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+beton 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+bocanci 4 doc10 doc3 doc7 doc9
+brad 6 doc0 doc10 doc11 doc7 doc8 doc9
+busola 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+cabana 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+cabanier 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+cald 6 doc0 doc10 doc11 doc3 doc7 doc9
+campie 6 doc0 doc10 doc3 doc5 doc7 doc9
+canale 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+ceai 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+ceata 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+comert 6 doc0 doc10 doc11 doc3 doc7 doc9
+cort 4 doc0 doc10 doc7 doc9
+cristal 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+curaj 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+curat 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+curent 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+dans 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+delta 4 doc0 doc10 doc11 doc7
+dezvoltare 6 doc0 doc10 doc3 doc5 doc7 doc8
+dig 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+distractie 6 doc0 doc11 doc3 doc7 doc8 doc9
+drum 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+echipa 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+ecou 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+efort 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+emotie 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+explorare 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+faleza 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+floare 6 doc0 doc11 doc3 doc5 doc7 doc8
+fluviu 6 doc10 doc3 doc5 doc7 doc8 doc9
+foc 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+formatiuni 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+frica 6 doc0 doc10 doc11 doc7 doc8 doc9
+frig 6 doc0 doc10 doc3 doc5 doc7 doc8
+gheata 6 doc0 doc10 doc11 doc7 doc8 doc9
+harta 6 doc0 doc10 doc11 doc5 doc7 doc9
+iarna 5 doc0 doc10 doc7 doc8 doc9
+indigo 6 doc0 doc11 doc3 doc7 doc8 doc9
+informatii 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+inot 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+insula 4 doc10 doc7 doc8 doc9
+intuneric 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+istorie 6 doc0 doc10 doc3 doc5 doc7 doc8
+izvor 5 doc0 doc3 doc5 doc7 doc9
+lanterna 5 doc0 doc10 doc7 doc8 doc9
+lemne 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+liniste 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+lume 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+lumina 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+lumini 6 doc0 doc10 doc11 doc5 doc7 doc9
+lup 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+mal 6 doc0 doc10 doc11 doc3 doc7 doc9
+mare 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+marfa 4 doc10 doc3 doc7 doc9
+marinari 5 doc10 doc11 doc7 doc8 doc9
+masina 5 doc10 doc3 doc5 doc7 doc9
+masini 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+mister 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+momeala 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+motor 6 doc10 doc11 doc3 doc5 doc7 doc9
+munte 6 doc0 doc10 doc3 doc7 doc8 doc9
+muzica 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+natura 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+nisip 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+noapte 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+nor 5 doc0 doc10 doc11 doc5 doc7
+nuca 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+oras 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+orientare 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+otel 6 doc0 doc10 doc11 doc3 doc5 doc7
+padure 6 doc0 doc10 doc11 doc3 doc7 doc9
+palmieri 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+pamant 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+parcare 5 doc0 doc10 doc3 doc7 doc9
+pasari 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+patine 6 doc0 doc10 doc5 doc7 doc8 doc9
+patura 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+peisaj 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+pelican 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+pescarusi 6 doc0 doc10 doc3 doc7 doc8 doc9
+pescuit 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+peste 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+pestera 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+picaturi 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+plaja 6 doc0 doc10 doc3 doc5 doc7 doc8
+plasa 5 doc0 doc11 doc5 doc7 doc9
+ploaie 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+pod 6 doc0 doc10 doc5 doc7 doc8 doc9
+poiana 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+port 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+povesti 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+poza 4 doc10 doc11 doc7 doc8
+poze 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+prapastie 6 doc0 doc10 doc3 doc7 doc8 doc9
+prieteni 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+rasarit 7 doc0 doc10 doc11 doc3 doc5 doc8 doc9
+rece 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+reflectie 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+relaxare 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+rucsac 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+salcie 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+sanie 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+sare 6 doc0 doc10 doc11 doc7 doc8 doc9
+schi 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+scoici 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+semineu 6 doc0 doc10 doc3 doc5 doc7 doc9
+sezlong 6 doc0 doc10 doc5 doc7 doc8 doc9
+soare 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+somn 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+sport 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+stanca 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+stele 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+sticla 5 doc10 doc11 doc3 doc7 doc9
+stuf 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+tacere 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+tehnologie 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+timp 6 doc10 doc11 doc3 doc5 doc7 doc9
+transport 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+traseu 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+trezire 6 doc0 doc10 doc3 doc5 doc7 doc9
+turist 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+umbra 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+umbrela 6 doc0 doc10 doc11 doc7 doc8 doc9
+undita 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+urs 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+valuri 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+vant 4 doc0 doc10 doc5 doc8
+vapor 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+vasle 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+viitor 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+vin 6 doc0 doc10 doc5 doc7 doc8 doc9
+violet 6 doc0 doc10 doc11 doc7 doc8 doc9
+vis 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+zapada 6 doc0 doc3 doc5 doc7 doc8 doc9
+zgomot 6 doc0 doc11 doc5 doc7 doc8 doc9
+4 doc10 doc11 doc7 doc8
 OK
+7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
 OK
+6 doc0 doc10 doc11 doc7 doc8 doc9
 OK
+adancime 6 doc0 doc11 doc5 doc7 doc8 doc9
+adrenalina 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+aer 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+afaceri 6 doc0 doc10 doc3 doc7 doc8 doc9
+aglomeratie 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+amintiri 6 doc10 doc3 doc5 doc7 doc8 doc9
+amurg 6 doc0 doc3 doc5 doc7 doc8 doc9
+ancora 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+apa 6 doc0 doc10 doc11 doc3 doc7 doc8
+aventura 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+bani 6 doc0 doc10 doc11 doc3 doc7 doc9
+barca 6 doc0 doc10 doc11 doc3 doc5 doc7
+barcagiu 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+baterii 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+beton 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+bocanci 4 doc10 doc3 doc7 doc9
+brad 6 doc0 doc10 doc11 doc7 doc8 doc9
+busola 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+cabana 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+cabanier 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+cald 6 doc0 doc10 doc11 doc3 doc7 doc9
+campie 6 doc0 doc10 doc3 doc5 doc7 doc9
+canale 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+ceai 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+ceata 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+comert 6 doc0 doc10 doc11 doc3 doc7 doc9
+cort 4 doc0 doc10 doc7 doc9
+cristal 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+curaj 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+curat 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+curent 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+dans 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+delta 4 doc0 doc10 doc11 doc7
+dezvoltare 6 doc0 doc10 doc3 doc5 doc7 doc8
+dig 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+distractie 6 doc0 doc11 doc3 doc7 doc8 doc9
+drum 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+echipa 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+ecou 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+efort 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+emotie 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+explorare 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+faleza 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+floare 6 doc0 doc11 doc3 doc5 doc7 doc8
+fluviu 6 doc10 doc3 doc5 doc7 doc8 doc9
+foc 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+formatiuni 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+frica 6 doc0 doc10 doc11 doc7 doc8 doc9
+frig 6 doc0 doc10 doc3 doc5 doc7 doc8
+gheata 6 doc0 doc10 doc11 doc7 doc8 doc9
+harta 6 doc0 doc10 doc11 doc5 doc7 doc9
+iarna 5 doc0 doc10 doc7 doc8 doc9
+indigo 6 doc0 doc11 doc3 doc7 doc8 doc9
+informatii 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+inot 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+insula 4 doc10 doc7 doc8 doc9
+intuneric 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+istorie 6 doc0 doc10 doc3 doc5 doc7 doc8
+izvor 5 doc0 doc3 doc5 doc7 doc9
+lanterna 5 doc0 doc10 doc7 doc8 doc9
+lemne 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+liniste 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+lume 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+lumina 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+lumini 6 doc0 doc10 doc11 doc5 doc7 doc9
+lup 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+mal 6 doc0 doc10 doc11 doc3 doc7 doc9
+mare 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+marfa 4 doc10 doc3 doc7 doc9
+marinari 5 doc10 doc11 doc7 doc8 doc9
+masina 5 doc10 doc3 doc5 doc7 doc9
+masini 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+mister 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+momeala 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+motor 6 doc10 doc11 doc3 doc5 doc7 doc9
+munte 6 doc0 doc10 doc3 doc7 doc8 doc9
+muzica 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+natura 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+nisip 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+noapte 7 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+nor 5 doc0 doc10 doc11 doc5 doc7
+nuca 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+oras 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+orientare 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+otel 6 doc0 doc10 doc11 doc3 doc5 doc7
+padure 6 doc0 doc10 doc11 doc3 doc7 doc9
+palmieri 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+pamant 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+parcare 5 doc0 doc10 doc3 doc7 doc9
+pasari 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+patine 6 doc0 doc10 doc5 doc7 doc8 doc9
+patura 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+peisaj 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+pelican 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+pescarusi 6 doc0 doc10 doc3 doc7 doc8 doc9
+pescuit 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+peste 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+pestera 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+picaturi 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+plaja 6 doc0 doc10 doc3 doc5 doc7 doc8
+plasa 5 doc0 doc11 doc5 doc7 doc9
+ploaie 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+pod 6 doc0 doc10 doc5 doc7 doc8 doc9
+poiana 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+port 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+povesti 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+poza 4 doc10 doc11 doc7 doc8
+poze 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+prapastie 6 doc0 doc10 doc3 doc7 doc8 doc9
+prieteni 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+rasarit 7 doc0 doc10 doc11 doc3 doc5 doc8 doc9
+rece 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+reflectie 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+relaxare 7 doc0 doc11 doc3 doc5 doc7 doc8 doc9
+rucsac 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+salcie 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+sanie 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+sare 6 doc0 doc10 doc11 doc7 doc8 doc9
+schi 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+scoici 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+semineu 6 doc0 doc10 doc3 doc5 doc7 doc9
+sezlong 6 doc0 doc10 doc5 doc7 doc8 doc9
+soare 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+somn 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+sport 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+stanca 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+stele 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+sticla 5 doc10 doc11 doc3 doc7 doc9
+stuf 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+tacere 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+tehnologie 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+timp 6 doc10 doc11 doc3 doc5 doc7 doc9
+transport 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+traseu 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+trezire 6 doc0 doc10 doc3 doc5 doc7 doc9
+turist 7 doc0 doc10 doc11 doc3 doc5 doc7 doc9
+umbra 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+umbrela 6 doc0 doc10 doc11 doc7 doc8 doc9
+undita 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+urs 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+valuri 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+vant 4 doc0 doc10 doc5 doc8
+vapor 7 doc0 doc10 doc3 doc5 doc7 doc8 doc9
+vasle 7 doc0 doc10 doc11 doc3 doc7 doc8 doc9
+viitor 8 doc0 doc10 doc11 doc3 doc5 doc7 doc8 doc9
+vin 6 doc0 doc10 doc5 doc7 doc8 doc9
+violet 6 doc0 doc10 doc11 doc7 doc8 doc9
+vis 7 doc0 doc10 doc11 doc5 doc7 doc8 doc9
+zapada 6 doc0 doc3 doc5 doc7 doc8 doc9
+zgomot 6 doc0 doc11 doc5 doc7 doc8 doc9
