--- temp.out.norm	2026-05-12 13:22:07.881009148 +0300
+++ tests/out/test19.out.norm	2026-05-12 13:22:07.882009143 +0300
@@ -11,7 +11,6 @@
 freedom 1 AdawaleMaster
 mentor 1 ArnoDorian
 paris 1 ArnoDorian
-pirate 1 AdawaleMaster
 OK
 ally 1 EliseDeLaSerre
 assassin 3 AdawaleMaster ArnoDorian ShaoJun
@@ -20,5 +19,4 @@
 freedom 1 AdawaleMaster
 mentor 1 ArnoDorian
 paris 1 ArnoDorian
-pirate 1 AdawaleMaster
 templar 1 EliseDeLaSerre
