--- temp.out.norm	2026-05-12 10:39:30.465900701 +0300
+++ tests/out/test38.out.norm	2026-05-12 10:39:30.467900675 +0300
@@ -4,3 +4,16 @@
 OK
 OK
 OK
+OK
+OK
+4 ChloeFrager ElenaFisher NathanDrake SamDrake
+1 NathanDrake
+4 ChloeFrager ElenaFisher NathanDrake SamDrake
+1 NathanDrake
+1 SamDrake
+adventurer 4 ChloeFrager ElenaFisher NathanDrake SamDrake
+hunter 1 NathanDrake
+journalist 1 ElenaFisher
+relic 1 NathanDrake
+thief 1 SamDrake
+treasure 1 NathanDrake
