--- temp.out.norm	2026-05-12 10:39:30.101905465 +0300
+++ tests/out/test19.out.norm	2026-05-12 10:39:30.103905439 +0300
@@ -3,3 +3,20 @@
 OK
 OK
 OK
+OK
+OK
+assassin 3 AdawaleMaster ArnoDorian ShaoJun
+brotherhood 2 ArnoDorian ShaoJun
+china 1 ShaoJun
+freedom 1 AdawaleMaster
+mentor 1 ArnoDorian
+paris 1 ArnoDorian
+OK
+ally 1 EliseDeLaSerre
+assassin 3 AdawaleMaster ArnoDorian ShaoJun
+brotherhood 2 ArnoDorian ShaoJun
+china 1 ShaoJun
+freedom 1 AdawaleMaster
+mentor 1 ArnoDorian
+paris 1 ArnoDorian
+templar 1 EliseDeLaSerre
