--- temp.out.norm	2026-05-15 16:28:39.321545731 +0300
+++ tests/out/test60.out.norm	2026-05-15 16:28:39.327912335 +0300
@@ -1085,4 +1085,15 @@
 turist 11 doc1 doc10 doc12 doc13 doc18 doc2 doc3 doc5 doc6 doc7 doc9
 umbra 15 doc0 doc1 doc12 doc13 doc15 doc16 doc17 doc18 doc19 doc2 doc21 doc6 doc7 doc8 doc9
 umbrela 16 doc0 doc1 doc10 doc12 doc13 doc15 doc16 doc18 doc19 doc2 doc21 doc3 doc4 doc6 doc7 doc9
-und
\ No newline at end of file
+undita 16 doc0 doc1 doc10 doc12 doc13 doc15 doc16 doc17 doc18 doc19 doc20 doc21 doc4 doc6 doc8 doc9
+urs 13 doc0 doc1 doc10 doc12 doc13 doc17 doc18 doc2 doc3 doc4 doc5 doc7 doc9
+valuri 15 doc0 doc1 doc12 doc16 doc17 doc19 doc2 doc20 doc21 doc3 doc4 doc6 doc7 doc8 doc9
+vant 16 doc0 doc1 doc10 doc13 doc15 doc16 doc17 doc18 doc19 doc20 doc3 doc4 doc6 doc7 doc8 doc9
+vapor 13 doc0 doc1 doc10 doc12 doc13 doc17 doc19 doc2 doc20 doc3 doc4 doc6 doc7
+vasle 16 doc0 doc1 doc10 doc12 doc13 doc15 doc16 doc17 doc18 doc19 doc2 doc3 doc6 doc7 doc8 doc9
+viitor 13 doc1 doc12 doc13 doc16 doc17 doc18 doc19 doc2 doc20 doc21 doc4 doc6 doc8
+vin 11 doc0 doc1 doc10 doc12 doc16 doc18 doc20 doc4 doc6 doc7 doc8
+violet 12 doc1 doc13 doc16 doc17 doc18 doc2 doc20 doc4 doc6 doc7 doc8 doc9
+vis 16 doc10 doc12 doc13 doc15 doc16 doc18 doc19 doc20 doc21 doc3 doc4 doc5 doc6 doc7 doc8 doc9
+zapada 14 doc12 doc13 doc16 doc18 doc19 doc2 doc20 doc21 doc3 doc4 doc5 doc7 doc8 doc9
+zgomot 16 doc0 doc1 doc10 doc12 doc13 doc15 doc16 doc17 doc18 doc20 doc3 doc5 doc6 doc7 doc8 doc9
